declare -x AZ_BATCH_ACCOUNT_NAME="exp1hildebatch"
declare -x AZ_BATCH_ACCOUNT_URL="https://exp1hildebatch.southeastasia.batch.azure.com/"
declare -x AZ_BATCH_CERTIFICATES_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00008/certs"
declare -x AZ_BATCH_JOB_ID="random_forest"
declare -x AZ_BATCH_JOB_PREP_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/jobpreparation"
declare -x AZ_BATCH_JOB_PREP_WORKING_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/jobpreparation/wd"
declare -x AZ_BATCH_NODE_ID="tvm-2112069986_1-20180411t200903z"
declare -x AZ_BATCH_NODE_IS_DEDICATED="true"
declare -x AZ_BATCH_NODE_ROOT_DIR="/mnt/batch/tasks"
declare -x AZ_BATCH_NODE_SHARED_DIR="/mnt/batch/tasks/shared"
declare -x AZ_BATCH_NODE_STARTUP_DIR="/mnt/batch/tasks/startup"
declare -x AZ_BATCH_POOL_ID="experiment"
declare -x AZ_BATCH_TASK_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00008"
declare -x AZ_BATCH_TASK_ID="task-00008"
declare -x AZ_BATCH_TASK_USER="_azbatch"
declare -x AZ_BATCH_TASK_USER_IDENTITY="PoolAdmin"
declare -x AZ_BATCH_TASK_WORKING_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00008/wd"
declare -x HOME="/mnt/batch/tasks/workitems/random_forest/job-1/task-00008/wd"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/batch/tasks/shared:/mnt/batch/tasks/workitems/random_forest/job-1/task-00008/wd"
declare -x PWD="/mnt/batch/tasks/workitems/random_forest/job-1/task-00008/wd"
declare -x SHLVL="1"
declare -x USER="_azbatch"
[2018-04-11 21:31:55] Results will not be logged locally.
[2018-04-11 21:31:55] Retrieving tasks...
[2018-04-11 21:31:55] Tasks will be retrieved based on provided task ids.
[2018-04-11 21:31:55] The following tasks were retrieved: [3021].
[2018-04-11 21:32:01] Downloaded task 3021.
[2018-04-11 21:32:01] Prepared pipelines.
[2018-04-11 21:32:01] Started run 1 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 2 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 3 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 4 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 5 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 6 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 7 on task 3021, dataset 'sick'.
[2018-04-11 21:32:01] Started run 8 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 2: Run already exists in server. Run id(s): {8956004}
[2018-04-11 21:32:06] Started run 9 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 1: Run already exists in server. Run id(s): {8956027}
[2018-04-11 21:32:06] Started run 10 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 3: Run already exists in server. Run id(s): {8955995}
[2018-04-11 21:32:06] Started run 11 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 4: Run already exists in server. Run id(s): {8956026}
[2018-04-11 21:32:06] Error in run 5: Run already exists in server. Run id(s): {8996093}
[2018-04-11 21:32:06] Started run 12 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Started run 13 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 7: Run already exists in server. Run id(s): {8996083}
[2018-04-11 21:32:06] Started run 14 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 6: Run already exists in server. Run id(s): {8955997}
[2018-04-11 21:32:06] Started run 15 on task 3021, dataset 'sick'.
[2018-04-11 21:32:06] Error in run 8: Run already exists in server. Run id(s): {8956038}
[2018-04-11 21:32:06] Started run 16 on task 3021, dataset 'sick'.
[2018-04-11 21:32:10] Error in run 9: Run already exists in server. Run id(s): {8955991}
[2018-04-11 21:32:10] Started run 17 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 10: Run already exists in server. Run id(s): {8996081}
[2018-04-11 21:32:11] Started run 18 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 11: Run already exists in server. Run id(s): {8956007}
[2018-04-11 21:32:11] Started run 19 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 12: Run already exists in server. Run id(s): {8956009}
[2018-04-11 21:32:11] Started run 20 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 13: Run already exists in server. Run id(s): {8956043}
[2018-04-11 21:32:11] Started run 21 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 15: Run already exists in server. Run id(s): {8996099}
[2018-04-11 21:32:11] Started run 22 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 14: Run already exists in server. Run id(s): {8956032}
[2018-04-11 21:32:11] Started run 23 on task 3021, dataset 'sick'.
[2018-04-11 21:32:11] Error in run 16: Run already exists in server. Run id(s): {8956044}
[2018-04-11 21:32:11] Started run 24 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Error in run 22: Run already exists in server. Run id(s): {8956025}
[2018-04-11 21:32:15] Error in run 18: Run already exists in server. Run id(s): {8956039}
[2018-04-11 21:32:15] Error in run 21: Run already exists in server. Run id(s): {8956006}
[2018-04-11 21:32:15] Error in run 23: Run already exists in server. Run id(s): {8956042}
[2018-04-11 21:32:15] Error in run 19: Run already exists in server. Run id(s): {8996096}
[2018-04-11 21:32:15] Started run 25 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Error in run 20: Run already exists in server. Run id(s): {8996110}
[2018-04-11 21:32:15] Error in run 17: Run already exists in server. Run id(s): {8996098}
[2018-04-11 21:32:15] Started run 29 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Started run 30 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Started run 27 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Started run 31 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Started run 26 on task 3021, dataset 'sick'.
[2018-04-11 21:32:15] Started run 28 on task 3021, dataset 'sick'.
[2018-04-11 21:32:16] Error in run 24: Run already exists in server. Run id(s): {8956046}
[2018-04-11 21:32:16] Started run 32 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Error in run 25: Run already exists in server. Run id(s): {8956008}
[2018-04-11 21:32:20] Started run 33 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Error in run 27: Run already exists in server. Run id(s): {8956028}
[2018-04-11 21:32:20] Started run 34 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Error in run 28: Run already exists in server. Run id(s): {8956036}
[2018-04-11 21:32:20] Started run 35 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Error in run 31: Run already exists in server. Run id(s): {8956045}
[2018-04-11 21:32:20] Error in run 26: Run already exists in server. Run id(s): {8956022}
[2018-04-11 21:32:20] Error in run 29: Run already exists in server. Run id(s): {8956019}
[2018-04-11 21:32:20] Error in run 30: Run already exists in server. Run id(s): {8956001}
[2018-04-11 21:32:20] Started run 36 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Started run 37 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Started run 39 on task 3021, dataset 'sick'.
[2018-04-11 21:32:20] Started run 38 on task 3021, dataset 'sick'.
[2018-04-11 21:32:21] Error in run 32: Run already exists in server. Run id(s): {8955987}
[2018-04-11 21:32:21] Started run 40 on task 3021, dataset 'sick'.
[2018-04-11 21:32:24] Error in run 33: Run already exists in server. Run id(s): {8955993}
[2018-04-11 21:32:24] Started run 41 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Error in run 36: Run already exists in server. Run id(s): {8996151}
[2018-04-11 21:32:25] Started run 42 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Error in run 37: Run already exists in server. Run id(s): {8996155}
[2018-04-11 21:32:25] Error in run 39: Run already exists in server. Run id(s): {8955992}
[2018-04-11 21:32:25] Error in run 38: Run already exists in server. Run id(s): {8956031}
[2018-04-11 21:32:25] Started run 43 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Error in run 35: Run already exists in server. Run id(s): {8956024}
[2018-04-11 21:32:25] Error in run 34: Run already exists in server. Run id(s): {8956017}
[2018-04-11 21:32:25] Started run 46 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Started run 47 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Started run 44 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Started run 45 on task 3021, dataset 'sick'.
[2018-04-11 21:32:25] Error in run 40: Run already exists in server. Run id(s): {8956034}
[2018-04-11 21:32:25] Started run 48 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Error in run 41: Run already exists in server. Run id(s): {8956037}
[2018-04-11 21:32:29] Started run 49 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Error in run 43: Run already exists in server. Run id(s): {8955982}
[2018-04-11 21:32:29] Error in run 46: Run already exists in server. Run id(s): {8956020}
[2018-04-11 21:32:29] Started run 50 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Error in run 42: Run already exists in server. Run id(s): {8996159}
[2018-04-11 21:32:29] Error in run 45: Run already exists in server. Run id(s): {8955980}
[2018-04-11 21:32:29] Error in run 44: Run already exists in server. Run id(s): {8956040}
[2018-04-11 21:32:29] Started run 51 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Error in run 47: Run already exists in server. Run id(s): {8956011}
[2018-04-11 21:32:29] Started run 54 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Started run 55 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Started run 53 on task 3021, dataset 'sick'.
[2018-04-11 21:32:29] Started run 52 on task 3021, dataset 'sick'.
[2018-04-11 21:32:30] Error in run 48: Run already exists in server. Run id(s): {8956030}
[2018-04-11 21:32:30] Started run 56 on task 3021, dataset 'sick'.
[2018-04-11 21:32:33] Error in run 49: Run already exists in server. Run id(s): {8996168}
[2018-04-11 21:32:33] Started run 57 on task 3021, dataset 'sick'.
[2018-04-11 21:32:34] Error in run 55: Run already exists in server. Run id(s): {8955999}
[2018-04-11 21:32:34] Error in run 51: Run already exists in server. Run id(s): {8956002}
[2018-04-11 21:32:34] Error in run 52: Run already exists in server. Run id(s): {8956033}
[2018-04-11 21:32:34] Error in run 54: Run already exists in server. Run id(s): {8996171}
[2018-04-11 21:32:34] Started run 58 on task 3021, dataset 'sick'.
[2018-04-11 21:32:34] Error in run 50: Run already exists in server. Run id(s): {8996158}
[2018-04-11 21:32:34] Error in run 53: Run already exists in server. Run id(s): {8956012}
[2018-04-11 21:32:34] Started run 59 on task 3021, dataset 'sick'.
[2018-04-11 21:32:34] Started run 62 on task 3021, dataset 'sick'.
[2018-04-11 21:32:34] Started run 60 on task 3021, dataset 'sick'.
[2018-04-11 21:32:34] Started run 63 on task 3021, dataset 'sick'.
[2018-04-11 21:32:34] Started run 61 on task 3021, dataset 'sick'.
[2018-04-11 21:32:35] Error in run 56: Run already exists in server. Run id(s): {8956048}
[2018-04-11 21:32:35] Started run 64 on task 3021, dataset 'sick'.
[2018-04-11 21:32:38] Error in run 57: Run already exists in server. Run id(s): {8956047}
[2018-04-11 21:32:38] Started run 65 on task 3021, dataset 'sick'.
[2018-04-11 21:32:39] Error in run 61: Run already exists in server. Run id(s): {8956018}
[2018-04-11 21:32:39] Error in run 59: Run already exists in server. Run id(s): {8955990}
[2018-04-11 21:32:39] Error in run 58: Run already exists in server. Run id(s): {8956003}
[2018-04-11 21:32:39] Started run 66 on task 3021, dataset 'sick'.
[2018-04-11 21:32:39] Error in run 62: Run already exists in server. Run id(s): {8996196}
[2018-04-11 21:32:39] Started run 68 on task 3021, dataset 'sick'.
[2018-04-11 21:32:39] Error in run 60: Run already exists in server. Run id(s): {8996185}
[2018-04-11 21:32:39] Error in run 63: Run already exists in server. Run id(s): {8956015}
[2018-04-11 21:32:39] Started run 69 on task 3021, dataset 'sick'.
[2018-04-11 21:32:39] Started run 67 on task 3021, dataset 'sick'.
[2018-04-11 21:32:39] Started run 70 on task 3021, dataset 'sick'.
[2018-04-11 21:32:39] Started run 71 on task 3021, dataset 'sick'.
[2018-04-11 21:32:40] Error in run 64: Run already exists in server. Run id(s): {8996212}
[2018-04-11 21:32:40] Started run 72 on task 3021, dataset 'sick'.
[2018-04-11 21:32:43] Error in run 65: Run already exists in server. Run id(s): {8956010}
[2018-04-11 21:32:43] Started run 73 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Error in run 70: Run already exists in server. Run id(s): {8956041}
[2018-04-11 21:32:44] Error in run 67: Run already exists in server. Run id(s): {8955988}
[2018-04-11 21:32:44] Error in run 71: Run already exists in server. Run id(s): {8955998}
[2018-04-11 21:32:44] Error in run 68: Run already exists in server. Run id(s): {8956000}
[2018-04-11 21:32:44] Started run 74 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Error in run 66: Run already exists in server. Run id(s): {8955996}
[2018-04-11 21:32:44] Error in run 69: Run already exists in server. Run id(s): {8996222}
[2018-04-11 21:32:44] Started run 75 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Started run 78 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Started run 77 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Started run 79 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Started run 76 on task 3021, dataset 'sick'.
[2018-04-11 21:32:44] Error in run 72: Run already exists in server. Run id(s): {8956021}
[2018-04-11 21:32:44] Started run 80 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Error in run 73: Run already exists in server. Run id(s): {8956054}
[2018-04-11 21:32:48] Started run 81 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Error in run 74: Run already exists in server. Run id(s): {8956052}
[2018-04-11 21:32:48] Started run 82 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Error in run 78: Run already exists in server. Run id(s): {8956058}
[2018-04-11 21:32:48] Error in run 77: Run already exists in server. Run id(s): {8956071}
[2018-04-11 21:32:48] Error in run 75: Run already exists in server. Run id(s): {8956059}
[2018-04-11 21:32:48] Started run 83 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Error in run 76: Run already exists in server. Run id(s): {8956064}
[2018-04-11 21:32:48] Started run 86 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Error in run 79: Run already exists in server. Run id(s): {8956056}
[2018-04-11 21:32:48] Started run 84 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Started run 87 on task 3021, dataset 'sick'.
[2018-04-11 21:32:48] Started run 85 on task 3021, dataset 'sick'.
[2018-04-11 21:32:49] Error in run 80: Run already exists in server. Run id(s): {8956053}
[2018-04-11 21:32:49] Started run 88 on task 3021, dataset 'sick'.
[2018-04-11 21:32:52] Error in run 81: Run already exists in server. Run id(s): {8956049}
[2018-04-11 21:32:52] Started run 89 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Error in run 82: Run already exists in server. Run id(s): {8956050}
[2018-04-11 21:32:53] Started run 90 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Error in run 84: Run already exists in server. Run id(s): {8956060}
[2018-04-11 21:32:53] Started run 91 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Error in run 83: Run already exists in server. Run id(s): {8956055}
[2018-04-11 21:32:53] Error in run 87: Run already exists in server. Run id(s): {8956086}
[2018-04-11 21:32:53] Error in run 85: Run already exists in server. Run id(s): {8956078}
[2018-04-11 21:32:53] Started run 92 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Error in run 88: Run already exists in server. Run id(s): {8956072}
[2018-04-11 21:32:53] Error in run 86: Run already exists in server. Run id(s): {8956070}
[2018-04-11 21:32:53] Started run 95 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Started run 96 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Started run 93 on task 3021, dataset 'sick'.
[2018-04-11 21:32:53] Started run 94 on task 3021, dataset 'sick'.
[2018-04-11 21:32:57] Error in run 89: Run already exists in server. Run id(s): {8956069}
[2018-04-11 21:32:57] Started run 97 on task 3021, dataset 'sick'.
[2018-04-11 21:32:57] Error in run 90: Run already exists in server. Run id(s): {8996229}
[2018-04-11 21:32:57] Started run 98 on task 3021, dataset 'sick'.
[2018-04-11 21:32:57] Error in run 91: Run already exists in server. Run id(s): {8996236}
[2018-04-11 21:32:57] Started run 99 on task 3021, dataset 'sick'.
[2018-04-11 21:32:58] Error in run 92: Run already exists in server. Run id(s): {8996243}
[2018-04-11 21:32:58] Started run 100 on task 3021, dataset 'sick'.
[2018-04-11 21:32:58] Error in run 96: Run already exists in server. Run id(s): {8996258}
[2018-04-11 21:32:58] Error in run 93: Run already exists in server. Run id(s): {8996248}
[2018-04-11 21:32:58] Error in run 95: Run already exists in server. Run id(s): {8996253}
[2018-04-11 21:32:58] Started run 101 on task 3021, dataset 'sick'.
[2018-04-11 21:32:58] Error in run 94: Run already exists in server. Run id(s): {8996234}
[2018-04-11 21:32:58] Started run 104 on task 3021, dataset 'sick'.
[2018-04-11 21:32:58] Started run 102 on task 3021, dataset 'sick'.
[2018-04-11 21:32:58] Started run 103 on task 3021, dataset 'sick'.
[2018-04-11 21:33:02] Error in run 98: Run already exists in server. Run id(s): {8996288}
[2018-04-11 21:33:02] Error in run 97: Run already exists in server. Run id(s): {8996282}
[2018-04-11 21:33:02] Started run 105 on task 3021, dataset 'sick'.
[2018-04-11 21:33:02] Started run 106 on task 3021, dataset 'sick'.
[2018-04-11 21:33:02] Error in run 99: Run already exists in server. Run id(s): {8996296}
[2018-04-11 21:33:02] Started run 107 on task 3021, dataset 'sick'.
[2018-04-11 21:33:03] Error in run 102: Run already exists in server. Run id(s): {8996309}
[2018-04-11 21:33:03] Error in run 101: Run already exists in server. Run id(s): {8996273}
[2018-04-11 21:33:03] Error in run 100: Run already exists in server. Run id(s): {8996299}
[2018-04-11 21:33:03] Error in run 104: Run already exists in server. Run id(s): {8996320}
[2018-04-11 21:33:03] Started run 108 on task 3021, dataset 'sick'.
[2018-04-11 21:33:03] Error in run 103: Run already exists in server. Run id(s): {8996314}
[2018-04-11 21:33:03] Started run 110 on task 3021, dataset 'sick'.
[2018-04-11 21:33:03] Started run 112 on task 3021, dataset 'sick'.
[2018-04-11 21:33:03] Started run 111 on task 3021, dataset 'sick'.
[2018-04-11 21:33:03] Started run 109 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Error in run 106: Run already exists in server. Run id(s): {8996346}
[2018-04-11 21:33:07] Error in run 105: Run already exists in server. Run id(s): {8996323}
[2018-04-11 21:33:07] Error in run 107: Run already exists in server. Run id(s): {8996354}
[2018-04-11 21:33:07] Started run 113 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Started run 114 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Started run 115 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Error in run 108: Run already exists in server. Run id(s): {8996352}
[2018-04-11 21:33:07] Started run 116 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Error in run 110: Run already exists in server. Run id(s): {8996357}
[2018-04-11 21:33:07] Started run 117 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Error in run 109: Run already exists in server. Run id(s): {8996345}
[2018-04-11 21:33:07] Error in run 111: Run already exists in server. Run id(s): {8996369}
[2018-04-11 21:33:07] Error in run 112: Run already exists in server. Run id(s): {8996363}
[2018-04-11 21:33:07] Started run 118 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Started run 119 on task 3021, dataset 'sick'.
[2018-04-11 21:33:07] Started run 120 on task 3021, dataset 'sick'.
[2018-04-11 21:33:11] Error in run 115: Run already exists in server. Run id(s): {8996403}
[2018-04-11 21:33:11] Error in run 114: Run already exists in server. Run id(s): {8996415}
[2018-04-11 21:33:11] Started run 121 on task 3021, dataset 'sick'.
[2018-04-11 21:33:11] Started run 122 on task 3021, dataset 'sick'.
[2018-04-11 21:33:11] Error in run 113: Run already exists in server. Run id(s): {8996374}
[2018-04-11 21:33:11] Started run 123 on task 3021, dataset 'sick'.
[2018-04-11 21:33:12] Error in run 116: Run already exists in server. Run id(s): {8996418}
[2018-04-11 21:33:12] Started run 124 on task 3021, dataset 'sick'.
[2018-04-11 21:33:12] Error in run 118: Run already exists in server. Run id(s): {8996422}
[2018-04-11 21:33:12] Error in run 117: Run already exists in server. Run id(s): {8996421}
[2018-04-11 21:33:12] Started run 125 on task 3021, dataset 'sick'.
[2018-04-11 21:33:12] Error in run 119: Run already exists in server. Run id(s): {8996428}
[2018-04-11 21:33:12] Error in run 120: Run already exists in server. Run id(s): {8996432}
[2018-04-11 21:33:12] Started run 127 on task 3021, dataset 'sick'.
[2018-04-11 21:33:12] Started run 128 on task 3021, dataset 'sick'.
[2018-04-11 21:33:12] Started run 126 on task 3021, dataset 'sick'.
[2018-04-11 21:33:16] Error in run 123: Run already exists in server. Run id(s): {8996464}
[2018-04-11 21:33:16] Error in run 121: Run already exists in server. Run id(s): {8996436}
[2018-04-11 21:33:16] Error in run 122: Run already exists in server. Run id(s): {8996474}
[2018-04-11 21:33:16] Started run 129 on task 3021, dataset 'sick'.
[2018-04-11 21:33:16] Started run 131 on task 3021, dataset 'sick'.
[2018-04-11 21:33:16] Error in run 124: Run already exists in server. Run id(s): {8996488}
[2018-04-11 21:33:16] Started run 132 on task 3021, dataset 'sick'.
[2018-04-11 21:33:16] Started run 130 on task 3021, dataset 'sick'.
[2018-04-11 21:33:17] Error in run 128: Run already exists in server. Run id(s): {8996494}
[2018-04-11 21:33:17] Error in run 125: Run already exists in server. Run id(s): {8996492}
[2018-04-11 21:33:17] Started run 133 on task 3021, dataset 'sick'.
[2018-04-11 21:33:17] Error in run 126: Run already exists in server. Run id(s): {8996482}
[2018-04-11 21:33:17] Error in run 127: Run already exists in server. Run id(s): {8996499}
[2018-04-11 21:33:17] Started run 135 on task 3021, dataset 'sick'.
[2018-04-11 21:33:17] Started run 136 on task 3021, dataset 'sick'.
[2018-04-11 21:33:17] Started run 134 on task 3021, dataset 'sick'.
[2018-04-11 21:33:21] Error in run 129: Run already exists in server. Run id(s): {8996491}
[2018-04-11 21:33:21] Error in run 131: Run already exists in server. Run id(s): {8996533}
[2018-04-11 21:33:21] Error in run 132: Run already exists in server. Run id(s): {8996556}
[2018-04-11 21:33:21] Error in run 130: Run already exists in server. Run id(s): {8996519}
[2018-04-11 21:33:21] Started run 137 on task 3021, dataset 'sick'.
[2018-04-11 21:33:21] Started run 138 on task 3021, dataset 'sick'.
[2018-04-11 21:33:21] Started run 139 on task 3021, dataset 'sick'.
[2018-04-11 21:33:21] Started run 140 on task 3021, dataset 'sick'.
[2018-04-11 21:33:22] Error in run 133: Run already exists in server. Run id(s): {8996561}
[2018-04-11 21:33:22] Error in run 135: Run already exists in server. Run id(s): {8996552}
[2018-04-11 21:33:22] Started run 141 on task 3021, dataset 'sick'.
[2018-04-11 21:33:22] Error in run 134: Run already exists in server. Run id(s): {8996559}
[2018-04-11 21:33:22] Error in run 136: Run already exists in server. Run id(s): {8996568}
[2018-04-11 21:33:22] Started run 143 on task 3021, dataset 'sick'.
[2018-04-11 21:33:22] Started run 144 on task 3021, dataset 'sick'.
[2018-04-11 21:33:22] Started run 142 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Error in run 137: Run already exists in server. Run id(s): {8996548}
[2018-04-11 21:33:26] Error in run 138: Run already exists in server. Run id(s): {8996553}
[2018-04-11 21:33:26] Error in run 140: Run already exists in server. Run id(s): {8996598}
[2018-04-11 21:33:26] Started run 145 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Error in run 139: Run already exists in server. Run id(s): {8996581}
[2018-04-11 21:33:26] Started run 146 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Started run 148 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Started run 147 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Error in run 141: Run already exists in server. Run id(s): {8996622}
[2018-04-11 21:33:26] Started run 149 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Error in run 144: Run already exists in server. Run id(s): {8996621}
[2018-04-11 21:33:26] Started run 150 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Error in run 142: Run already exists in server. Run id(s): {8996624}
[2018-04-11 21:33:26] Error in run 143: Run already exists in server. Run id(s): {8996615}
[2018-04-11 21:33:26] Started run 151 on task 3021, dataset 'sick'.
[2018-04-11 21:33:26] Started run 152 on task 3021, dataset 'sick'.
[2018-04-11 21:33:30] Error in run 147: Run already exists in server. Run id(s): {8996612}
[2018-04-11 21:33:30] Error in run 148: Run already exists in server. Run id(s): {8996659}
[2018-04-11 21:33:30] Started run 153 on task 3021, dataset 'sick'.
[2018-04-11 21:33:30] Error in run 145: Run already exists in server. Run id(s): {8996639}
[2018-04-11 21:33:30] Error in run 146: Run already exists in server. Run id(s): {8996607}
[2018-04-11 21:33:30] Started run 155 on task 3021, dataset 'sick'.
[2018-04-11 21:33:30] Started run 156 on task 3021, dataset 'sick'.
[2018-04-11 21:33:30] Started run 154 on task 3021, dataset 'sick'.
[2018-04-11 21:33:31] Error in run 149: Run already exists in server. Run id(s): {8996656}
[2018-04-11 21:33:31] Started run 157 on task 3021, dataset 'sick'.
[2018-04-11 21:33:31] Error in run 152: Run already exists in server. Run id(s): {8996675}
[2018-04-11 21:33:31] Started run 158 on task 3021, dataset 'sick'.
[2018-04-11 21:33:31] Error in run 150: Run already exists in server. Run id(s): {8996680}
[2018-04-11 21:33:31] Error in run 151: Run already exists in server. Run id(s): {8996662}
[2018-04-11 21:33:31] Started run 159 on task 3021, dataset 'sick'.
[2018-04-11 21:33:31] Started run 160 on task 3021, dataset 'sick'.
[2018-04-11 21:33:35] Error in run 153: Run already exists in server. Run id(s): {8996690}
[2018-04-11 21:33:35] Error in run 154: Run already exists in server. Run id(s): {8996670}
[2018-04-11 21:33:35] Error in run 155: Run already exists in server. Run id(s): {8996696}
[2018-04-11 21:33:35] Started run 161 on task 3021, dataset 'sick'.
[2018-04-11 21:33:35] Error in run 157: Run already exists in server. Run id(s): {8996722}
[2018-04-11 21:33:35] Error in run 156: Run already exists in server. Run id(s): {8996714}
[2018-04-11 21:33:35] Started run 162 on task 3021, dataset 'sick'.
[2018-04-11 21:33:35] Started run 164 on task 3021, dataset 'sick'.
[2018-04-11 21:33:35] Started run 163 on task 3021, dataset 'sick'.
[2018-04-11 21:33:35] Started run 165 on task 3021, dataset 'sick'.
[2018-04-11 21:33:36] Error in run 158: Run already exists in server. Run id(s): {8996723}
[2018-04-11 21:33:36] Error in run 159: Run already exists in server. Run id(s): {8996726}
[2018-04-11 21:33:36] Started run 166 on task 3021, dataset 'sick'.
[2018-04-11 21:33:36] Error in run 160: Run already exists in server. Run id(s): {8996724}
[2018-04-11 21:33:36] Started run 167 on task 3021, dataset 'sick'.
[2018-04-11 21:33:36] Started run 168 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Error in run 161: Run already exists in server. Run id(s): {8996725}
[2018-04-11 21:33:40] Started run 169 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Error in run 162: Run already exists in server. Run id(s): {8996746}
[2018-04-11 21:33:40] Started run 170 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Error in run 163: Run already exists in server. Run id(s): {8996752}
[2018-04-11 21:33:40] Error in run 165: Run already exists in server. Run id(s): {8996779}
[2018-04-11 21:33:40] Error in run 164: Run already exists in server. Run id(s): {8996782}
[2018-04-11 21:33:40] Started run 171 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Started run 173 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Started run 172 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Error in run 166: Run already exists in server. Run id(s): {8996804}
[2018-04-11 21:33:40] Started run 174 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Error in run 167: Run already exists in server. Run id(s): {8996793}
[2018-04-11 21:33:40] Started run 175 on task 3021, dataset 'sick'.
[2018-04-11 21:33:40] Error in run 168: Run already exists in server. Run id(s): {8996796}
[2018-04-11 21:33:40] Started run 176 on task 3021, dataset 'sick'.
[2018-04-11 21:33:44] Error in run 169: Run already exists in server. Run id(s): {8996805}
[2018-04-11 21:33:44] Started run 177 on task 3021, dataset 'sick'.
[2018-04-11 21:33:44] Error in run 170: Run already exists in server. Run id(s): {8996789}
[2018-04-11 21:33:44] Started run 178 on task 3021, dataset 'sick'.
[2018-04-11 21:33:45] Error in run 174: Run already exists in server. Run id(s): {8996852}
[2018-04-11 21:33:45] Error in run 171: Run already exists in server. Run id(s): {8996807}
[2018-04-11 21:33:45] Error in run 172: Run already exists in server. Run id(s): {8996854}
[2018-04-11 21:33:45] Started run 179 on task 3021, dataset 'sick'.
[2018-04-11 21:33:45] Started run 181 on task 3021, dataset 'sick'.
[2018-04-11 21:33:45] Error in run 173: Run already exists in server. Run id(s): {8996847}
[2018-04-11 21:33:45] Started run 182 on task 3021, dataset 'sick'.
[2018-04-11 21:33:45] Error in run 175: Run already exists in server. Run id(s): {8996857}
[2018-04-11 21:33:45] Started run 183 on task 3021, dataset 'sick'.
[2018-04-11 21:33:45] Started run 180 on task 3021, dataset 'sick'.
[2018-04-11 21:33:45] Error in run 176: Run already exists in server. Run id(s): {8996872}
[2018-04-11 21:33:45] Started run 184 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Error in run 177: Run already exists in server. Run id(s): {8956068}
[2018-04-11 21:33:49] Started run 185 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Error in run 178: Run already exists in server. Run id(s): {8996884}
[2018-04-11 21:33:49] Started run 186 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Error in run 179: Run already exists in server. Run id(s): {8996858}
[2018-04-11 21:33:49] Started run 187 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Error in run 181: Run already exists in server. Run id(s): {8996920}
[2018-04-11 21:33:49] Started run 188 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Error in run 183: Run already exists in server. Run id(s): {8956115}
[2018-04-11 21:33:49] Error in run 180: Run already exists in server. Run id(s): {8996870}
[2018-04-11 21:33:49] Error in run 182: Run already exists in server. Run id(s): {8996915}
[2018-04-11 21:33:49] Started run 189 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Error in run 184: Run already exists in server. Run id(s): {8956079}
[2018-04-11 21:33:49] Started run 192 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Started run 190 on task 3021, dataset 'sick'.
[2018-04-11 21:33:49] Started run 191 on task 3021, dataset 'sick'.
[2018-04-11 21:33:53] Error in run 185: Run already exists in server. Run id(s): {8956084}
[2018-04-11 21:33:53] Error in run 186: Run already exists in server. Run id(s): {8956138}
[2018-04-11 21:33:53] Started run 193 on task 3021, dataset 'sick'.
[2018-04-11 21:33:53] Started run 194 on task 3021, dataset 'sick'.
[2018-04-11 21:33:54] Error in run 187: Run already exists in server. Run id(s): {8996919}
[2018-04-11 21:33:54] Error in run 188: Run already exists in server. Run id(s): {8996932}
[2018-04-11 21:33:54] Started run 195 on task 3021, dataset 'sick'.
[2018-04-11 21:33:54] Started run 196 on task 3021, dataset 'sick'.
[2018-04-11 21:33:54] Error in run 192: Run already exists in server. Run id(s): {8996927}
[2018-04-11 21:33:54] Started run 197 on task 3021, dataset 'sick'.
[2018-04-11 21:33:54] Error in run 191: Run already exists in server. Run id(s): {8996918}
[2018-04-11 21:33:54] Started run 198 on task 3021, dataset 'sick'.
[2018-04-11 21:33:54] Error in run 190: Run already exists in server. Run id(s): {8996899}
[2018-04-11 21:33:54] Error in run 189: Run already exists in server. Run id(s): {8996921}
[2018-04-11 21:33:54] Started run 199 on task 3021, dataset 'sick'.
[2018-04-11 21:33:54] Started run 200 on task 3021, dataset 'sick'.
[2018-04-11 21:33:58] Error in run 194: Run already exists in server. Run id(s): {8956085}
[2018-04-11 21:33:58] Started run 201 on task 3021, dataset 'sick'.
[2018-04-11 21:33:58] Error in run 193: Run already exists in server. Run id(s): {8956095}
[2018-04-11 21:33:58] Started run 202 on task 3021, dataset 'sick'.
[2018-04-11 21:33:58] Error in run 196: Run already exists in server. Run id(s): {8996985}
[2018-04-11 21:33:58] Error in run 195: Run already exists in server. Run id(s): {8996969}
[2018-04-11 21:33:58] Started run 203 on task 3021, dataset 'sick'.
[2018-04-11 21:33:58] Started run 204 on task 3021, dataset 'sick'.
[2018-04-11 21:33:58] Error in run 197: Run already exists in server. Run id(s): {8956075}
[2018-04-11 21:33:58] Started run 205 on task 3021, dataset 'sick'.
[2018-04-11 21:33:59] Error in run 198: Run already exists in server. Run id(s): {8996974}
[2018-04-11 21:33:59] Started run 206 on task 3021, dataset 'sick'.
[2018-04-11 21:33:59] Error in run 199: Run already exists in server. Run id(s): {8996990}
[2018-04-11 21:33:59] Started run 207 on task 3021, dataset 'sick'.
[2018-04-11 21:33:59] Error in run 200: Run already exists in server. Run id(s): {8996978}
[2018-04-11 21:33:59] Started run 208 on task 3021, dataset 'sick'.
[2018-04-11 21:34:02] Error in run 202: Run already exists in server. Run id(s): {8996980}
[2018-04-11 21:34:02] Error in run 201: Run already exists in server. Run id(s): {8996992}
[2018-04-11 21:34:02] Started run 209 on task 3021, dataset 'sick'.
[2018-04-11 21:34:02] Started run 210 on task 3021, dataset 'sick'.
[2018-04-11 21:34:03] Error in run 203: Run already exists in server. Run id(s): {8996989}
[2018-04-11 21:34:03] Started run 211 on task 3021, dataset 'sick'.
[2018-04-11 21:34:03] Error in run 204: Run already exists in server. Run id(s): {8997018}
[2018-04-11 21:34:03] Error in run 205: Run already exists in server. Run id(s): {8997035}
[2018-04-11 21:34:03] Started run 212 on task 3021, dataset 'sick'.
[2018-04-11 21:34:03] Started run 213 on task 3021, dataset 'sick'.
[2018-04-11 21:34:03] Error in run 206: Run already exists in server. Run id(s): {8997048}
[2018-04-11 21:34:03] Started run 214 on task 3021, dataset 'sick'.
[2018-04-11 21:34:03] Error in run 207: Run already exists in server. Run id(s): {8997038}
[2018-04-11 21:34:03] Started run 215 on task 3021, dataset 'sick'.
[2018-04-11 21:34:03] Error in run 208: Run already exists in server. Run id(s): {8997051}
[2018-04-11 21:34:03] Started run 216 on task 3021, dataset 'sick'.
[2018-04-11 21:34:07] Error in run 209: Run already exists in server. Run id(s): {8997043}
[2018-04-11 21:34:07] Started run 217 on task 3021, dataset 'sick'.
[2018-04-11 21:34:07] Error in run 210: Run already exists in server. Run id(s): {8997016}
[2018-04-11 21:34:07] Started run 218 on task 3021, dataset 'sick'.
[2018-04-11 21:34:07] Error in run 211: Run already exists in server. Run id(s): {8997050}
[2018-04-11 21:34:07] Started run 219 on task 3021, dataset 'sick'.
[2018-04-11 21:34:07] Error in run 212: Run already exists in server. Run id(s): {8997060}
[2018-04-11 21:34:07] Started run 220 on task 3021, dataset 'sick'.
[2018-04-11 21:34:08] Error in run 215: Run already exists in server. Run id(s): {8997072}
[2018-04-11 21:34:08] Started run 221 on task 3021, dataset 'sick'.
[2018-04-11 21:34:08] Error in run 213: Run already exists in server. Run id(s): {8956088}
[2018-04-11 21:34:08] Error in run 214: Run already exists in server. Run id(s): {8997083}
[2018-04-11 21:34:08] Started run 222 on task 3021, dataset 'sick'.
[2018-04-11 21:34:08] Started run 223 on task 3021, dataset 'sick'.
[2018-04-11 21:34:08] Error in run 216: Run already exists in server. Run id(s): {8956112}
[2018-04-11 21:34:08] Started run 224 on task 3021, dataset 'sick'.
[2018-04-11 21:34:11] Error in run 217: Run already exists in server. Run id(s): {8956106}
[2018-04-11 21:34:11] Started run 225 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 218: Run already exists in server. Run id(s): {8997095}
[2018-04-11 21:34:12] Started run 226 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 219: Run already exists in server. Run id(s): {8997093}
[2018-04-11 21:34:12] Started run 227 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 220: Run already exists in server. Run id(s): {8956097}
[2018-04-11 21:34:12] Started run 228 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 221: Run already exists in server. Run id(s): {8997077}
[2018-04-11 21:34:12] Started run 229 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 222: Run already exists in server. Run id(s): {8956087}
[2018-04-11 21:34:12] Started run 230 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 224: Run already exists in server. Run id(s): {8956103}
[2018-04-11 21:34:12] Started run 231 on task 3021, dataset 'sick'.
[2018-04-11 21:34:12] Error in run 223: Run already exists in server. Run id(s): {8956111}
[2018-04-11 21:34:12] Started run 232 on task 3021, dataset 'sick'.
[2018-04-11 21:34:16] Error in run 225: Run already exists in server. Run id(s): {8956073}
[2018-04-11 21:34:16] Started run 233 on task 3021, dataset 'sick'.
[2018-04-11 21:34:16] Error in run 226: Run already exists in server. Run id(s): {8997102}
[2018-04-11 21:34:16] Error in run 228: Run already exists in server. Run id(s): {8956113}
[2018-04-11 21:34:16] Started run 234 on task 3021, dataset 'sick'.
[2018-04-11 21:34:16] Error in run 227: Run already exists in server. Run id(s): {8956123}
[2018-04-11 21:34:16] Started run 235 on task 3021, dataset 'sick'.
[2018-04-11 21:34:16] Started run 236 on task 3021, dataset 'sick'.
[2018-04-11 21:34:17] Error in run 229: Run already exists in server. Run id(s): {8956108}
[2018-04-11 21:34:17] Started run 237 on task 3021, dataset 'sick'.
[2018-04-11 21:34:17] Error in run 230: Run already exists in server. Run id(s): {8956118}
[2018-04-11 21:34:17] Started run 238 on task 3021, dataset 'sick'.
[2018-04-11 21:34:17] Error in run 231: Run already exists in server. Run id(s): {8956098}
[2018-04-11 21:34:17] Error in run 232: Run already exists in server. Run id(s): {8956074}
[2018-04-11 21:34:17] Started run 239 on task 3021, dataset 'sick'.
[2018-04-11 21:34:17] Started run 240 on task 3021, dataset 'sick'.
[2018-04-11 21:34:20] Error in run 233: Run already exists in server. Run id(s): {8956117}
[2018-04-11 21:34:20] Started run 241 on task 3021, dataset 'sick'.
[2018-04-11 21:34:21] Error in run 234: Run already exists in server. Run id(s): {8997133}
[2018-04-11 21:34:21] Error in run 235: Run already exists in server. Run id(s): {8956099}
[2018-04-11 21:34:21] Started run 242 on task 3021, dataset 'sick'.
[2018-04-11 21:34:21] Started run 243 on task 3021, dataset 'sick'.
[2018-04-11 21:34:21] Error in run 236: Run already exists in server. Run id(s): {8956107}
[2018-04-11 21:34:21] Started run 244 on task 3021, dataset 'sick'.
[2018-04-11 21:34:21] Error in run 237: Run already exists in server. Run id(s): {8956066}
[2018-04-11 21:34:21] Started run 245 on task 3021, dataset 'sick'.
[2018-04-11 21:34:21] Error in run 238: Run already exists in server. Run id(s): {8956120}
[2018-04-11 21:34:21] Started run 246 on task 3021, dataset 'sick'.
[2018-04-11 21:34:22] Error in run 239: Run already exists in server. Run id(s): {8956109}
[2018-04-11 21:34:22] Error in run 240: Run already exists in server. Run id(s): {8956129}
[2018-04-11 21:34:22] Started run 247 on task 3021, dataset 'sick'.
[2018-04-11 21:34:22] Started run 248 on task 3021, dataset 'sick'.
[2018-04-11 21:34:25] Error in run 241: Run already exists in server. Run id(s): {8956089}
[2018-04-11 21:34:25] Started run 249 on task 3021, dataset 'sick'.
[2018-04-11 21:34:25] Error in run 242: Run already exists in server. Run id(s): {8997148}
[2018-04-11 21:34:25] Started run 250 on task 3021, dataset 'sick'.
[2018-04-11 21:34:25] Error in run 243: Run already exists in server. Run id(s): {8956100}
[2018-04-11 21:34:25] Started run 251 on task 3021, dataset 'sick'.
[2018-04-11 21:34:26] Error in run 244: Run already exists in server. Run id(s): {8956127}
[2018-04-11 21:34:26] Started run 252 on task 3021, dataset 'sick'.
[2018-04-11 21:34:26] Error in run 245: Run already exists in server. Run id(s): {8956131}
[2018-04-11 21:34:26] Started run 253 on task 3021, dataset 'sick'.
[2018-04-11 21:34:26] Error in run 246: Run already exists in server. Run id(s): {8956122}
[2018-04-11 21:34:26] Started run 254 on task 3021, dataset 'sick'.
[2018-04-11 21:34:26] Error in run 247: Run already exists in server. Run id(s): {8997146}
[2018-04-11 21:34:26] Started run 255 on task 3021, dataset 'sick'.
[2018-04-11 21:34:27] Error in run 248: Run already exists in server. Run id(s): {8956125}
[2018-04-11 21:34:27] Started run 256 on task 3021, dataset 'sick'.
[2018-04-11 21:34:30] Error in run 250: Run already exists in server. Run id(s): {8956135}
[2018-04-11 21:34:30] Error in run 249: Run already exists in server. Run id(s): {8956092}
[2018-04-11 21:34:30] Started run 257 on task 3021, dataset 'sick'.
[2018-04-11 21:34:30] Started run 258 on task 3021, dataset 'sick'.
[2018-04-11 21:34:30] Error in run 251: Run already exists in server. Run id(s): {8956094}
[2018-04-11 21:34:30] Started run 259 on task 3021, dataset 'sick'.
[2018-04-11 21:34:30] Error in run 252: Run already exists in server. Run id(s): {8956091}
[2018-04-11 21:34:30] Started run 260 on task 3021, dataset 'sick'.
[2018-04-11 21:34:31] Error in run 254: Run already exists in server. Run id(s): {8956137}
[2018-04-11 21:34:31] Error in run 253: Run already exists in server. Run id(s): {8997155}
[2018-04-11 21:34:31] Started run 261 on task 3021, dataset 'sick'.
[2018-04-11 21:34:31] Started run 262 on task 3021, dataset 'sick'.
[2018-04-11 21:34:31] Error in run 255: Run already exists in server. Run id(s): {8956102}
[2018-04-11 21:34:31] Started run 263 on task 3021, dataset 'sick'.
[2018-04-11 21:34:31] Error in run 256: Run already exists in server. Run id(s): {8997161}
[2018-04-11 21:34:31] Started run 264 on task 3021, dataset 'sick'.
[2018-04-11 21:34:34] Error in run 258: Run already exists in server. Run id(s): {8956119}
[2018-04-11 21:34:34] Error in run 259: Run already exists in server. Run id(s): {8956136}
[2018-04-11 21:34:34] Started run 265 on task 3021, dataset 'sick'.
[2018-04-11 21:34:34] Started run 266 on task 3021, dataset 'sick'.
[2018-04-11 21:34:34] Error in run 257: Run already exists in server. Run id(s): {8956096}
[2018-04-11 21:34:34] Started run 267 on task 3021, dataset 'sick'.
[2018-04-11 21:34:35] Error in run 260: Run already exists in server. Run id(s): {8956082}
[2018-04-11 21:34:35] Started run 268 on task 3021, dataset 'sick'.
[2018-04-11 21:34:35] Error in run 261: Run already exists in server. Run id(s): {8997163}
[2018-04-11 21:34:35] Started run 269 on task 3021, dataset 'sick'.
[2018-04-11 21:34:35] Error in run 263: Run already exists in server. Run id(s): {8956142}
[2018-04-11 21:34:35] Error in run 262: Run already exists in server. Run id(s): {8956104}
[2018-04-11 21:34:35] Started run 270 on task 3021, dataset 'sick'.
[2018-04-11 21:34:35] Started run 271 on task 3021, dataset 'sick'.
[2018-04-11 21:34:36] Error in run 264: Run already exists in server. Run id(s): {8956133}
[2018-04-11 21:34:36] Started run 272 on task 3021, dataset 'sick'.
[2018-04-11 21:34:39] Error in run 265: Run already exists in server. Run id(s): {8997170}
[2018-04-11 21:34:39] Started run 273 on task 3021, dataset 'sick'.
[2018-04-11 21:34:39] Error in run 267: Run already exists in server. Run id(s): {8956124}
[2018-04-11 21:34:39] Started run 274 on task 3021, dataset 'sick'.
[2018-04-11 21:34:39] Error in run 266: Run already exists in server. Run id(s): {8956134}
[2018-04-11 21:34:39] Started run 275 on task 3021, dataset 'sick'.
[2018-04-11 21:34:39] Error in run 268: Run already exists in server. Run id(s): {8956090}
[2018-04-11 21:34:39] Started run 276 on task 3021, dataset 'sick'.
[2018-04-11 21:34:40] Error in run 269: Run already exists in server. Run id(s): {8956141}
[2018-04-11 21:34:40] Started run 277 on task 3021, dataset 'sick'.
[2018-04-11 21:34:40] Error in run 270: Run already exists in server. Run id(s): {8997186}
[2018-04-11 21:34:40] Started run 278 on task 3021, dataset 'sick'.
[2018-04-11 21:34:40] Error in run 271: Run already exists in server. Run id(s): {8956132}
[2018-04-11 21:34:40] Started run 279 on task 3021, dataset 'sick'.
[2018-04-11 21:34:40] Error in run 272: Run already exists in server. Run id(s): {8956128}
[2018-04-11 21:34:40] Started run 280 on task 3021, dataset 'sick'.
[2018-04-11 21:34:43] Error in run 273: Run already exists in server. Run id(s): {8956139}
[2018-04-11 21:34:43] Started run 281 on task 3021, dataset 'sick'.
[2018-04-11 21:34:43] Error in run 274: Run already exists in server. Run id(s): {8956110}
[2018-04-11 21:34:43] Started run 282 on task 3021, dataset 'sick'.
[2018-04-11 21:34:43] Error in run 275: Run already exists in server. Run id(s): {8997190}
[2018-04-11 21:34:43] Started run 283 on task 3021, dataset 'sick'.
[2018-04-11 21:34:44] Error in run 276: Run already exists in server. Run id(s): {8956145}
[2018-04-11 21:34:44] Started run 284 on task 3021, dataset 'sick'.
[2018-04-11 21:34:44] Error in run 277: Run already exists in server. Run id(s): {8997196}
[2018-04-11 21:34:44] Started run 285 on task 3021, dataset 'sick'.
[2018-04-11 21:34:45] Error in run 278: Run already exists in server. Run id(s): {8956126}
[2018-04-11 21:34:45] Started run 286 on task 3021, dataset 'sick'.
[2018-04-11 21:34:45] Error in run 280: Run already exists in server. Run id(s): {8956155}
[2018-04-11 21:34:45] Error in run 279: Run already exists in server. Run id(s): {8956158}
[2018-04-11 21:34:45] Started run 287 on task 3021, dataset 'sick'.
[2018-04-11 21:34:45] Started run 288 on task 3021, dataset 'sick'.
[2018-04-11 21:34:48] Error in run 281: Run already exists in server. Run id(s): {8956156}
[2018-04-11 21:34:48] Started run 289 on task 3021, dataset 'sick'.
[2018-04-11 21:34:48] Error in run 283: Run already exists in server. Run id(s): {8956160}
[2018-04-11 21:34:48] Error in run 282: Run already exists in server. Run id(s): {8956154}
[2018-04-11 21:34:48] Started run 290 on task 3021, dataset 'sick'.
[2018-04-11 21:34:48] Started run 291 on task 3021, dataset 'sick'.
[2018-04-11 21:34:49] Error in run 284: Run already exists in server. Run id(s): {8956144}
[2018-04-11 21:34:49] Started run 292 on task 3021, dataset 'sick'.
[2018-04-11 21:34:49] Error in run 285: Run already exists in server. Run id(s): {8956149}
[2018-04-11 21:34:49] Started run 293 on task 3021, dataset 'sick'.
[2018-04-11 21:34:49] Error in run 286: Run already exists in server. Run id(s): {8956151}
[2018-04-11 21:34:49] Started run 294 on task 3021, dataset 'sick'.
[2018-04-11 21:34:50] Error in run 287: Run already exists in server. Run id(s): {8956147}
[2018-04-11 21:34:50] Started run 295 on task 3021, dataset 'sick'.
[2018-04-11 21:34:50] Error in run 288: Run already exists in server. Run id(s): {8956148}
[2018-04-11 21:34:50] Started run 296 on task 3021, dataset 'sick'.
[2018-04-11 21:34:52] Error in run 289: Run already exists in server. Run id(s): {8956152}
[2018-04-11 21:34:52] Started run 297 on task 3021, dataset 'sick'.
[2018-04-11 21:34:52] Error in run 290: Run already exists in server. Run id(s): {8956167}
[2018-04-11 21:34:52] Started run 298 on task 3021, dataset 'sick'.
[2018-04-11 21:34:53] Error in run 291: Run already exists in server. Run id(s): {8956146}
[2018-04-11 21:34:53] Started run 299 on task 3021, dataset 'sick'.
[2018-04-11 21:34:53] Error in run 292: Run already exists in server. Run id(s): {8956162}
[2018-04-11 21:34:53] Started run 300 on task 3021, dataset 'sick'.
[2018-04-11 21:34:53] Error in run 293: Run already exists in server. Run id(s): {8956153}
[2018-04-11 21:34:53] Started run 301 on task 3021, dataset 'sick'.
[2018-04-11 21:34:54] Error in run 294: Run already exists in server. Run id(s): {8956159}
[2018-04-11 21:34:54] Started run 302 on task 3021, dataset 'sick'.
[2018-04-11 21:34:54] Error in run 295: Run already exists in server. Run id(s): {8956192}
[2018-04-11 21:34:54] Started run 303 on task 3021, dataset 'sick'.
[2018-04-11 21:34:54] Error in run 296: Run already exists in server. Run id(s): {8956169}
[2018-04-11 21:34:54] Started run 304 on task 3021, dataset 'sick'.
[2018-04-11 21:34:57] Error in run 297: Run already exists in server. Run id(s): {8956188}
[2018-04-11 21:34:57] Started run 305 on task 3021, dataset 'sick'.
[2018-04-11 21:34:57] Error in run 298: Run already exists in server. Run id(s): {8956165}
[2018-04-11 21:34:57] Started run 306 on task 3021, dataset 'sick'.
[2018-04-11 21:34:57] Error in run 299: Run already exists in server. Run id(s): {8956182}
[2018-04-11 21:34:57] Started run 307 on task 3021, dataset 'sick'.
[2018-04-11 21:34:58] Error in run 300: Run already exists in server. Run id(s): {8956185}
[2018-04-11 21:34:58] Started run 308 on task 3021, dataset 'sick'.
[2018-04-11 21:34:58] Error in run 301: Run already exists in server. Run id(s): {8956168}
[2018-04-11 21:34:58] Started run 309 on task 3021, dataset 'sick'.
[2018-04-11 21:34:58] Error in run 302: Run already exists in server. Run id(s): {8956175}
[2018-04-11 21:34:58] Started run 310 on task 3021, dataset 'sick'.
[2018-04-11 21:34:58] Error in run 303: Run already exists in server. Run id(s): {8956170}
[2018-04-11 21:34:58] Started run 311 on task 3021, dataset 'sick'.
[2018-04-11 21:34:59] Error in run 304: Run already exists in server. Run id(s): {8956180}
[2018-04-11 21:34:59] Started run 312 on task 3021, dataset 'sick'.
[2018-04-11 21:35:01] Error in run 305: Run already exists in server. Run id(s): {8956197}
[2018-04-11 21:35:01] Started run 313 on task 3021, dataset 'sick'.
[2018-04-11 21:35:01] Error in run 306: Run already exists in server. Run id(s): {8997233}
[2018-04-11 21:35:01] Started run 314 on task 3021, dataset 'sick'.
[2018-04-11 21:35:02] Error in run 307: Run already exists in server. Run id(s): {8997230}
[2018-04-11 21:35:02] Started run 315 on task 3021, dataset 'sick'.
[2018-04-11 21:35:02] Error in run 308: Run already exists in server. Run id(s): {8997224}
[2018-04-11 21:35:02] Error in run 309: Run already exists in server. Run id(s): {8997244}
[2018-04-11 21:35:02] Started run 316 on task 3021, dataset 'sick'.
[2018-04-11 21:35:02] Started run 317 on task 3021, dataset 'sick'.
[2018-04-11 21:35:03] Error in run 310: Run already exists in server. Run id(s): {8997241}
[2018-04-11 21:35:03] Started run 318 on task 3021, dataset 'sick'.
[2018-04-11 21:35:03] Error in run 312: Run already exists in server. Run id(s): {8997256}
[2018-04-11 21:35:03] Error in run 311: Run already exists in server. Run id(s): {8997239}
[2018-04-11 21:35:03] Started run 319 on task 3021, dataset 'sick'.
[2018-04-11 21:35:03] Started run 320 on task 3021, dataset 'sick'.
[2018-04-11 21:35:06] Error in run 313: Run already exists in server. Run id(s): {8997247}
[2018-04-11 21:35:06] Started run 321 on task 3021, dataset 'sick'.
[2018-04-11 21:35:06] Error in run 315: Run already exists in server. Run id(s): {8956172}
[2018-04-11 21:35:06] Started run 322 on task 3021, dataset 'sick'.
[2018-04-11 21:35:06] Error in run 314: Run already exists in server. Run id(s): {8997271}
[2018-04-11 21:35:06] Started run 323 on task 3021, dataset 'sick'.
[2018-04-11 21:35:07] Error in run 317: Run already exists in server. Run id(s): {8997289}
[2018-04-11 21:35:07] Started run 324 on task 3021, dataset 'sick'.
[2018-04-11 21:35:07] Error in run 316: Run already exists in server. Run id(s): {8997294}
[2018-04-11 21:35:07] Started run 325 on task 3021, dataset 'sick'.
[2018-04-11 21:35:07] Error in run 318: Run already exists in server. Run id(s): {8997301}
[2018-04-11 21:35:07] Started run 326 on task 3021, dataset 'sick'.
[2018-04-11 21:35:08] Error in run 319: Run already exists in server. Run id(s): {8956193}
[2018-04-11 21:35:08] Started run 327 on task 3021, dataset 'sick'.
[2018-04-11 21:35:08] Error in run 320: Run already exists in server. Run id(s): {8997290}
[2018-04-11 21:35:08] Started run 328 on task 3021, dataset 'sick'.
[2018-04-11 21:35:10] Error in run 321: Run already exists in server. Run id(s): {8997302}
[2018-04-11 21:35:10] Started run 329 on task 3021, dataset 'sick'.
[2018-04-11 21:35:11] Error in run 322: Run already exists in server. Run id(s): {8997310}
[2018-04-11 21:35:11] Started run 330 on task 3021, dataset 'sick'.
[2018-04-11 21:35:11] Error in run 323: Run already exists in server. Run id(s): {8997292}
[2018-04-11 21:35:11] Started run 331 on task 3021, dataset 'sick'.
[2018-04-11 21:35:12] Error in run 324: Run already exists in server. Run id(s): {8997315}
[2018-04-11 21:35:12] Started run 332 on task 3021, dataset 'sick'.
[2018-04-11 21:35:12] Error in run 325: Run already exists in server. Run id(s): {8997326}
[2018-04-11 21:35:12] Started run 333 on task 3021, dataset 'sick'.
[2018-04-11 21:35:12] Error in run 326: Run already exists in server. Run id(s): {8997346}
[2018-04-11 21:35:12] Started run 334 on task 3021, dataset 'sick'.
[2018-04-11 21:35:12] Error in run 327: Run already exists in server. Run id(s): {8997330}
[2018-04-11 21:35:12] Started run 335 on task 3021, dataset 'sick'.
[2018-04-11 21:35:13] Error in run 328: Run already exists in server. Run id(s): {8997335}
[2018-04-11 21:35:13] Started run 336 on task 3021, dataset 'sick'.
[2018-04-11 21:35:15] Error in run 329: Run already exists in server. Run id(s): {8997355}
[2018-04-11 21:35:15] Started run 337 on task 3021, dataset 'sick'.
[2018-04-11 21:35:15] Error in run 330: Run already exists in server. Run id(s): {8997358}
[2018-04-11 21:35:15] Started run 338 on task 3021, dataset 'sick'.
[2018-04-11 21:35:15] Error in run 331: Run already exists in server. Run id(s): {8997349}
[2018-04-11 21:35:15] Started run 339 on task 3021, dataset 'sick'.
[2018-04-11 21:35:16] Error in run 332: Run already exists in server. Run id(s): {8997362}
[2018-04-11 21:35:16] Started run 340 on task 3021, dataset 'sick'.
[2018-04-11 21:35:16] Error in run 333: Run already exists in server. Run id(s): {8997387}
[2018-04-11 21:35:16] Started run 341 on task 3021, dataset 'sick'.
[2018-04-11 21:35:17] Error in run 334: Run already exists in server. Run id(s): {8997383}
[2018-04-11 21:35:17] Started run 342 on task 3021, dataset 'sick'.
[2018-04-11 21:35:17] Error in run 335: Run already exists in server. Run id(s): {8956187}
[2018-04-11 21:35:17] Started run 343 on task 3021, dataset 'sick'.
[2018-04-11 21:35:17] Error in run 336: Run already exists in server. Run id(s): {8956229}
[2018-04-11 21:35:17] Started run 344 on task 3021, dataset 'sick'.
[2018-04-11 21:35:20] Error in run 337: Run already exists in server. Run id(s): {8956206}
[2018-04-11 21:35:20] Error in run 338: Run already exists in server. Run id(s): {8997391}
[2018-04-11 21:35:20] Started run 345 on task 3021, dataset 'sick'.
[2018-04-11 21:35:20] Started run 346 on task 3021, dataset 'sick'.
[2018-04-11 21:35:20] Error in run 339: Run already exists in server. Run id(s): {8997389}
[2018-04-11 21:35:20] Started run 347 on task 3021, dataset 'sick'.
[2018-04-11 21:35:20] Error in run 340: Run already exists in server. Run id(s): {8997405}
[2018-04-11 21:35:20] Started run 348 on task 3021, dataset 'sick'.
[2018-04-11 21:35:21] Error in run 341: Run already exists in server. Run id(s): {8956221}
[2018-04-11 21:35:21] Started run 349 on task 3021, dataset 'sick'.
[2018-04-11 21:35:21] Error in run 342: Run already exists in server. Run id(s): {8956177}
[2018-04-11 21:35:21] Started run 350 on task 3021, dataset 'sick'.
[2018-04-11 21:35:21] Error in run 343: Run already exists in server. Run id(s): {8997409}
[2018-04-11 21:35:21] Started run 351 on task 3021, dataset 'sick'.
[2018-04-11 21:35:22] Error in run 344: Run already exists in server. Run id(s): {8956217}
[2018-04-11 21:35:22] Started run 352 on task 3021, dataset 'sick'.
[2018-04-11 21:35:24] Error in run 345: Run already exists in server. Run id(s): {8997390}
[2018-04-11 21:35:24] Started run 353 on task 3021, dataset 'sick'.
[2018-04-11 21:35:24] Error in run 346: Run already exists in server. Run id(s): {8997424}
[2018-04-11 21:35:24] Started run 354 on task 3021, dataset 'sick'.
[2018-04-11 21:35:25] Error in run 347: Run already exists in server. Run id(s): {8997432}
[2018-04-11 21:35:25] Started run 355 on task 3021, dataset 'sick'.
[2018-04-11 21:35:25] Error in run 349: Run already exists in server. Run id(s): {8956195}
[2018-04-11 21:35:25] Error in run 348: Run already exists in server. Run id(s): {8956219}
[2018-04-11 21:35:25] Started run 356 on task 3021, dataset 'sick'.
[2018-04-11 21:35:25] Started run 357 on task 3021, dataset 'sick'.
[2018-04-11 21:35:25] Error in run 350: Run already exists in server. Run id(s): {8956209}
[2018-04-11 21:35:25] Started run 358 on task 3021, dataset 'sick'.
[2018-04-11 21:35:26] Error in run 351: Run already exists in server. Run id(s): {8997436}
[2018-04-11 21:35:26] Started run 359 on task 3021, dataset 'sick'.
[2018-04-11 21:35:26] Error in run 352: Run already exists in server. Run id(s): {8956198}
[2018-04-11 21:35:26] Started run 360 on task 3021, dataset 'sick'.
[2018-04-11 21:35:29] Error in run 354: Run already exists in server. Run id(s): {8956216}
[2018-04-11 21:35:29] Error in run 353: Run already exists in server. Run id(s): {8956202}
[2018-04-11 21:35:29] Started run 361 on task 3021, dataset 'sick'.
[2018-04-11 21:35:29] Started run 362 on task 3021, dataset 'sick'.
[2018-04-11 21:35:29] Error in run 355: Run already exists in server. Run id(s): {8956231}
[2018-04-11 21:35:29] Started run 363 on task 3021, dataset 'sick'.
[2018-04-11 21:35:30] Error in run 356: Run already exists in server. Run id(s): {8956223}
[2018-04-11 21:35:30] Started run 364 on task 3021, dataset 'sick'.
[2018-04-11 21:35:30] Error in run 357: Run already exists in server. Run id(s): {8956161}
[2018-04-11 21:35:30] Started run 365 on task 3021, dataset 'sick'.
[2018-04-11 21:35:30] Error in run 358: Run already exists in server. Run id(s): {8956191}
[2018-04-11 21:35:30] Started run 366 on task 3021, dataset 'sick'.
[2018-04-11 21:35:30] Error in run 359: Run already exists in server. Run id(s): {8956208}
[2018-04-11 21:35:30] Started run 367 on task 3021, dataset 'sick'.
[2018-04-11 21:35:31] Error in run 360: Run already exists in server. Run id(s): {8956228}
[2018-04-11 21:35:31] Started run 368 on task 3021, dataset 'sick'.
[2018-04-11 21:35:33] Error in run 361: Run already exists in server. Run id(s): {8956179}
[2018-04-11 21:35:33] Started run 369 on task 3021, dataset 'sick'.
[2018-04-11 21:35:33] Error in run 362: Run already exists in server. Run id(s): {8956218}
[2018-04-11 21:35:33] Started run 370 on task 3021, dataset 'sick'.
[2018-04-11 21:35:34] Error in run 363: Run already exists in server. Run id(s): {8956237}
[2018-04-11 21:35:34] Started run 371 on task 3021, dataset 'sick'.
[2018-04-11 21:35:34] Error in run 364: Run already exists in server. Run id(s): {8956201}
[2018-04-11 21:35:34] Error in run 365: Run already exists in server. Run id(s): {8956181}
[2018-04-11 21:35:34] Started run 372 on task 3021, dataset 'sick'.
[2018-04-11 21:35:34] Started run 373 on task 3021, dataset 'sick'.
[2018-04-11 21:35:35] Error in run 367: Run already exists in server. Run id(s): {8956214}
[2018-04-11 21:35:35] Error in run 366: Run already exists in server. Run id(s): {8956222}
[2018-04-11 21:35:35] Started run 374 on task 3021, dataset 'sick'.
[2018-04-11 21:35:35] Started run 375 on task 3021, dataset 'sick'.
[2018-04-11 21:35:35] Error in run 368: Run already exists in server. Run id(s): {8956205}
[2018-04-11 21:35:35] Started run 376 on task 3021, dataset 'sick'.
[2018-04-11 21:35:38] Error in run 369: Run already exists in server. Run id(s): {8956200}
[2018-04-11 21:35:38] Error in run 370: Run already exists in server. Run id(s): {8956224}
[2018-04-11 21:35:38] Started run 377 on task 3021, dataset 'sick'.
[2018-04-11 21:35:38] Started run 378 on task 3021, dataset 'sick'.
[2018-04-11 21:35:38] Error in run 371: Run already exists in server. Run id(s): {8956176}
[2018-04-11 21:35:38] Started run 379 on task 3021, dataset 'sick'.
[2018-04-11 21:35:39] Error in run 372: Run already exists in server. Run id(s): {8956220}
[2018-04-11 21:35:39] Started run 380 on task 3021, dataset 'sick'.
[2018-04-11 21:35:39] Error in run 373: Run already exists in server. Run id(s): {8956215}
[2018-04-11 21:35:39] Started run 381 on task 3021, dataset 'sick'.
[2018-04-11 21:35:39] Error in run 375: Run already exists in server. Run id(s): {8956230}
[2018-04-11 21:35:39] Started run 382 on task 3021, dataset 'sick'.
[2018-04-11 21:35:39] Error in run 374: Run already exists in server. Run id(s): {8956234}
[2018-04-11 21:35:39] Started run 383 on task 3021, dataset 'sick'.
[2018-04-11 21:35:40] Error in run 376: Run already exists in server. Run id(s): {8956212}
[2018-04-11 21:35:40] Started run 384 on task 3021, dataset 'sick'.
[2018-04-11 21:35:43] Error in run 378: Run already exists in server. Run id(s): {8956183}
[2018-04-11 21:35:43] Error in run 379: Run already exists in server. Run id(s): {8956233}
[2018-04-11 21:35:43] Started run 385 on task 3021, dataset 'sick'.
[2018-04-11 21:35:43] Error in run 377: Run already exists in server. Run id(s): {8956232}
[2018-04-11 21:35:43] Started run 386 on task 3021, dataset 'sick'.
[2018-04-11 21:35:43] Started run 387 on task 3021, dataset 'sick'.
[2018-04-11 21:35:43] Error in run 381: Run already exists in server. Run id(s): {8956251}
[2018-04-11 21:35:43] Started run 388 on task 3021, dataset 'sick'.
[2018-04-11 21:35:44] Error in run 380: Run already exists in server. Run id(s): {8956203}
[2018-04-11 21:35:44] Started run 389 on task 3021, dataset 'sick'.
[2018-04-11 21:35:44] Error in run 382: Run already exists in server. Run id(s): {8956174}
[2018-04-11 21:35:44] Started run 390 on task 3021, dataset 'sick'.
[2018-04-11 21:35:44] Error in run 383: Run already exists in server. Run id(s): {8956211}
[2018-04-11 21:35:44] Started run 391 on task 3021, dataset 'sick'.
[2018-04-11 21:35:44] Error in run 384: Run already exists in server. Run id(s): {8956227}
[2018-04-11 21:35:44] Started run 392 on task 3021, dataset 'sick'.
[2018-04-11 21:35:47] Error in run 385: Run already exists in server. Run id(s): {8956244}
[2018-04-11 21:35:47] Started run 393 on task 3021, dataset 'sick'.
[2018-04-11 21:35:47] Error in run 387: Run already exists in server. Run id(s): {8956210}
[2018-04-11 21:35:47] Error in run 386: Run already exists in server. Run id(s): {8956225}
[2018-04-11 21:35:47] Started run 394 on task 3021, dataset 'sick'.
[2018-04-11 21:35:47] Started run 395 on task 3021, dataset 'sick'.
[2018-04-11 21:35:48] Error in run 388: Run already exists in server. Run id(s): {8956240}
[2018-04-11 21:35:48] Started run 396 on task 3021, dataset 'sick'.
[2018-04-11 21:35:48] Error in run 389: Run already exists in server. Run id(s): {8956236}
[2018-04-11 21:35:48] Started run 397 on task 3021, dataset 'sick'.
[2018-04-11 21:35:48] Error in run 390: Run already exists in server. Run id(s): {8956243}
[2018-04-11 21:35:48] Started run 398 on task 3021, dataset 'sick'.
[2018-04-11 21:35:49] Error in run 391: Run already exists in server. Run id(s): {8956253}
[2018-04-11 21:35:49] Started run 399 on task 3021, dataset 'sick'.
[2018-04-11 21:35:49] Error in run 392: Run already exists in server. Run id(s): {8956190}
[2018-04-11 21:35:49] Started run 400 on task 3021, dataset 'sick'.
[2018-04-11 21:35:52] Error in run 393: Run already exists in server. Run id(s): {8956199}
[2018-04-11 21:35:52] Started run 401 on task 3021, dataset 'sick'.
[2018-04-11 21:35:52] Error in run 395: Run already exists in server. Run id(s): {8956207}
[2018-04-11 21:35:52] Started run 402 on task 3021, dataset 'sick'.
[2018-04-11 21:35:52] Error in run 394: Run already exists in server. Run id(s): {8956204}
[2018-04-11 21:35:52] Started run 403 on task 3021, dataset 'sick'.
[2018-04-11 21:35:52] Error in run 396: Run already exists in server. Run id(s): {8956258}
[2018-04-11 21:35:52] Started run 404 on task 3021, dataset 'sick'.
[2018-04-11 21:35:53] Error in run 397: Run already exists in server. Run id(s): {8956248}
[2018-04-11 21:35:53] Started run 405 on task 3021, dataset 'sick'.
[2018-04-11 21:35:53] Error in run 398: Run already exists in server. Run id(s): {8956252}
[2018-04-11 21:35:53] Started run 406 on task 3021, dataset 'sick'.
[2018-04-11 21:35:53] Error in run 399: Run already exists in server. Run id(s): {8956249}
[2018-04-11 21:35:53] Started run 407 on task 3021, dataset 'sick'.
[2018-04-11 21:35:54] Error in run 400: Run already exists in server. Run id(s): {8997485}
[2018-04-11 21:35:54] Started run 408 on task 3021, dataset 'sick'.
[2018-04-11 21:35:56] Error in run 401: Run already exists in server. Run id(s): {8956250}
[2018-04-11 21:35:56] Started run 409 on task 3021, dataset 'sick'.
[2018-04-11 21:35:56] Error in run 402: Run already exists in server. Run id(s): {8956260}
[2018-04-11 21:35:56] Started run 410 on task 3021, dataset 'sick'.
[2018-04-11 21:35:56] Error in run 403: Run already exists in server. Run id(s): {8956239}
[2018-04-11 21:35:56] Started run 411 on task 3021, dataset 'sick'.
[2018-04-11 21:35:57] Error in run 404: Run already exists in server. Run id(s): {8956265}
[2018-04-11 21:35:57] Started run 412 on task 3021, dataset 'sick'.
[2018-04-11 21:35:57] Error in run 405: Run already exists in server. Run id(s): {8956256}
[2018-04-11 21:35:57] Started run 413 on task 3021, dataset 'sick'.
[2018-04-11 21:35:58] Error in run 406: Run already exists in server. Run id(s): {8956259}
[2018-04-11 21:35:58] Started run 414 on task 3021, dataset 'sick'.
[2018-04-11 21:35:58] Error in run 407: Run already exists in server. Run id(s): {8956242}
[2018-04-11 21:35:58] Started run 415 on task 3021, dataset 'sick'.
[2018-04-11 21:35:58] Error in run 408: Run already exists in server. Run id(s): {8956254}
[2018-04-11 21:35:58] Started run 416 on task 3021, dataset 'sick'.
[2018-04-11 21:36:01] Error in run 409: Run already exists in server. Run id(s): {8956271}
[2018-04-11 21:36:01] Started run 417 on task 3021, dataset 'sick'.
[2018-04-11 21:36:01] Error in run 410: Run already exists in server. Run id(s): {8956255}
[2018-04-11 21:36:01] Error in run 411: Run already exists in server. Run id(s): {8956272}
[2018-04-11 21:36:01] Started run 418 on task 3021, dataset 'sick'.
[2018-04-11 21:36:01] Started run 419 on task 3021, dataset 'sick'.
[2018-04-11 21:36:01] Error in run 412: Run already exists in server. Run id(s): {8956246}
[2018-04-11 21:36:01] Started run 420 on task 3021, dataset 'sick'.
[2018-04-11 21:36:02] Error in run 413: Run already exists in server. Run id(s): {8956266}
[2018-04-11 21:36:02] Started run 421 on task 3021, dataset 'sick'.
[2018-04-11 21:36:02] Error in run 414: Run already exists in server. Run id(s): {8956268}
[2018-04-11 21:36:02] Started run 422 on task 3021, dataset 'sick'.
[2018-04-11 21:36:03] Error in run 415: Run already exists in server. Run id(s): {8956275}
[2018-04-11 21:36:03] Started run 423 on task 3021, dataset 'sick'.
[2018-04-11 21:36:03] Error in run 416: Run already exists in server. Run id(s): {8956261}
[2018-04-11 21:36:03] Started run 424 on task 3021, dataset 'sick'.
[2018-04-11 21:36:05] Error in run 417: Run already exists in server. Run id(s): {8997479}
[2018-04-11 21:36:05] Started run 425 on task 3021, dataset 'sick'.
[2018-04-11 21:36:06] Error in run 419: Run already exists in server. Run id(s): {8956264}
[2018-04-11 21:36:06] Started run 426 on task 3021, dataset 'sick'.
[2018-04-11 21:36:06] Error in run 418: Run already exists in server. Run id(s): {8956263}
[2018-04-11 21:36:06] Started run 427 on task 3021, dataset 'sick'.
[2018-04-11 21:36:06] Error in run 420: Run already exists in server. Run id(s): {8956278}
[2018-04-11 21:36:06] Started run 428 on task 3021, dataset 'sick'.
[2018-04-11 21:36:06] Error in run 421: Run already exists in server. Run id(s): {8956285}
[2018-04-11 21:36:06] Started run 429 on task 3021, dataset 'sick'.
[2018-04-11 21:36:07] Error in run 422: Run already exists in server. Run id(s): {8956262}
[2018-04-11 21:36:07] Started run 430 on task 3021, dataset 'sick'.
[2018-04-11 21:36:07] Error in run 423: Run already exists in server. Run id(s): {8956269}
[2018-04-11 21:36:07] Error in run 424: Run already exists in server. Run id(s): {8956281}
[2018-04-11 21:36:07] Started run 431 on task 3021, dataset 'sick'.
[2018-04-11 21:36:07] Started run 432 on task 3021, dataset 'sick'.
[2018-04-11 21:36:10] Error in run 425: Run already exists in server. Run id(s): {8956282}
[2018-04-11 21:36:10] Started run 433 on task 3021, dataset 'sick'.
[2018-04-11 21:36:10] Error in run 426: Run already exists in server. Run id(s): {8956277}
[2018-04-11 21:36:10] Started run 434 on task 3021, dataset 'sick'.
[2018-04-11 21:36:10] Error in run 427: Run already exists in server. Run id(s): {8956280}
[2018-04-11 21:36:10] Started run 435 on task 3021, dataset 'sick'.
[2018-04-11 21:36:10] Error in run 428: Run already exists in server. Run id(s): {8956291}
[2018-04-11 21:36:10] Started run 436 on task 3021, dataset 'sick'.
[2018-04-11 21:36:11] Error in run 429: Run already exists in server. Run id(s): {8956297}
[2018-04-11 21:36:11] Started run 437 on task 3021, dataset 'sick'.
[2018-04-11 21:36:11] Error in run 430: Run already exists in server. Run id(s): {8956279}
[2018-04-11 21:36:11] Started run 438 on task 3021, dataset 'sick'.
[2018-04-11 21:36:12] Error in run 431: Run already exists in server. Run id(s): {8956301}
[2018-04-11 21:36:12] Started run 439 on task 3021, dataset 'sick'.
[2018-04-11 21:36:12] Error in run 432: Run already exists in server. Run id(s): {8997499}
[2018-04-11 21:36:12] Started run 440 on task 3021, dataset 'sick'.
[2018-04-11 21:36:14] Error in run 433: Run already exists in server. Run id(s): {8997510}
[2018-04-11 21:36:14] Started run 441 on task 3021, dataset 'sick'.
[2018-04-11 21:36:14] Error in run 434: Run already exists in server. Run id(s): {8956290}
[2018-04-11 21:36:14] Started run 442 on task 3021, dataset 'sick'.
[2018-04-11 21:36:15] Error in run 435: Run already exists in server. Run id(s): {8956286}
[2018-04-11 21:36:15] Error in run 436: Run already exists in server. Run id(s): {8997513}
[2018-04-11 21:36:15] Started run 443 on task 3021, dataset 'sick'.
[2018-04-11 21:36:15] Started run 444 on task 3021, dataset 'sick'.
[2018-04-11 21:36:15] Error in run 437: Run already exists in server. Run id(s): {8997516}
[2018-04-11 21:36:15] Started run 445 on task 3021, dataset 'sick'.
[2018-04-11 21:36:16] Error in run 438: Run already exists in server. Run id(s): {8997508}
[2018-04-11 21:36:16] Started run 446 on task 3021, dataset 'sick'.
[2018-04-11 21:36:16] Error in run 439: Run already exists in server. Run id(s): {8997523}
[2018-04-11 21:36:16] Started run 447 on task 3021, dataset 'sick'.
[2018-04-11 21:36:17] Error in run 440: Run already exists in server. Run id(s): {8997533}
[2018-04-11 21:36:17] Started run 448 on task 3021, dataset 'sick'.
[2018-04-11 21:36:19] Error in run 441: Run already exists in server. Run id(s): {8997529}
[2018-04-11 21:36:19] Started run 449 on task 3021, dataset 'sick'.
[2018-04-11 21:36:19] Error in run 442: Run already exists in server. Run id(s): {8997525}
[2018-04-11 21:36:19] Started run 450 on task 3021, dataset 'sick'.
[2018-04-11 21:36:20] Error in run 444: Run already exists in server. Run id(s): {8997566}
[2018-04-11 21:36:20] Started run 451 on task 3021, dataset 'sick'.
[2018-04-11 21:36:20] Error in run 443: Run already exists in server. Run id(s): {8997553}
[2018-04-11 21:36:20] Started run 452 on task 3021, dataset 'sick'.
[2018-04-11 21:36:20] Error in run 445: Run already exists in server. Run id(s): {8997557}
[2018-04-11 21:36:20] Started run 453 on task 3021, dataset 'sick'.
[2018-04-11 21:36:21] Error in run 446: Run already exists in server. Run id(s): {8997570}
[2018-04-11 21:36:21] Started run 454 on task 3021, dataset 'sick'.
[2018-04-11 21:36:21] Error in run 447: Run already exists in server. Run id(s): {8997571}
[2018-04-11 21:36:21] Started run 455 on task 3021, dataset 'sick'.
[2018-04-11 21:36:21] Error in run 448: Run already exists in server. Run id(s): {8997575}
[2018-04-11 21:36:21] Started run 456 on task 3021, dataset 'sick'.
[2018-04-11 21:36:24] Error in run 449: Run already exists in server. Run id(s): {8997587}
[2018-04-11 21:36:24] Started run 457 on task 3021, dataset 'sick'.
[2018-04-11 21:36:24] Error in run 450: Run already exists in server. Run id(s): {8997573}
[2018-04-11 21:36:24] Started run 458 on task 3021, dataset 'sick'.
[2018-04-11 21:36:24] Error in run 452: Run already exists in server. Run id(s): {8997619}
[2018-04-11 21:36:24] Error in run 451: Run already exists in server. Run id(s): {8956292}
[2018-04-11 21:36:24] Started run 459 on task 3021, dataset 'sick'.
[2018-04-11 21:36:24] Started run 460 on task 3021, dataset 'sick'.
[2018-04-11 21:36:25] Error in run 453: Run already exists in server. Run id(s): {8997618}
[2018-04-11 21:36:25] Started run 461 on task 3021, dataset 'sick'.
[2018-04-11 21:36:25] Error in run 455: Run already exists in server. Run id(s): {8997614}
[2018-04-11 21:36:25] Started run 462 on task 3021, dataset 'sick'.
[2018-04-11 21:36:25] Error in run 454: Run already exists in server. Run id(s): {8997615}
[2018-04-11 21:36:25] Started run 463 on task 3021, dataset 'sick'.
[2018-04-11 21:36:26] Error in run 456: Run already exists in server. Run id(s): {8997600}
[2018-04-11 21:36:26] Started run 464 on task 3021, dataset 'sick'.
[2018-04-11 21:36:28] Error in run 457: Run already exists in server. Run id(s): {8956318}
[2018-04-11 21:36:28] Started run 465 on task 3021, dataset 'sick'.
[2018-04-11 21:36:29] Error in run 458: Run already exists in server. Run id(s): {8997624}
[2018-04-11 21:36:29] Started run 466 on task 3021, dataset 'sick'.
[2018-04-11 21:36:29] Error in run 459: Run already exists in server. Run id(s): {8997636}
[2018-04-11 21:36:29] Error in run 460: Run already exists in server. Run id(s): {8997632}
[2018-04-11 21:36:29] Started run 467 on task 3021, dataset 'sick'.
[2018-04-11 21:36:29] Started run 468 on task 3021, dataset 'sick'.
[2018-04-11 21:36:29] Error in run 461: Run already exists in server. Run id(s): {8956307}
[2018-04-11 21:36:29] Started run 469 on task 3021, dataset 'sick'.
[2018-04-11 21:36:30] Error in run 463: Run already exists in server. Run id(s): {8956300}
[2018-04-11 21:36:30] Error in run 462: Run already exists in server. Run id(s): {8997658}
[2018-04-11 21:36:30] Started run 470 on task 3021, dataset 'sick'.
[2018-04-11 21:36:30] Started run 471 on task 3021, dataset 'sick'.
[2018-04-11 21:36:30] Error in run 464: Run already exists in server. Run id(s): {8997664}
[2018-04-11 21:36:30] Started run 472 on task 3021, dataset 'sick'.
[2018-04-11 21:36:33] Error in run 465: Run already exists in server. Run id(s): {8956309}
[2018-04-11 21:36:33] Started run 473 on task 3021, dataset 'sick'.
[2018-04-11 21:36:33] Error in run 466: Run already exists in server. Run id(s): {8956333}
[2018-04-11 21:36:33] Started run 474 on task 3021, dataset 'sick'.
[2018-04-11 21:36:33] Error in run 468: Run already exists in server. Run id(s): {8997666}
[2018-04-11 21:36:33] Error in run 467: Run already exists in server. Run id(s): {8997674}
[2018-04-11 21:36:33] Started run 475 on task 3021, dataset 'sick'.
[2018-04-11 21:36:33] Started run 476 on task 3021, dataset 'sick'.
[2018-04-11 21:36:34] Error in run 469: Run already exists in server. Run id(s): {8956339}
[2018-04-11 21:36:34] Started run 477 on task 3021, dataset 'sick'.
[2018-04-11 21:36:34] Error in run 470: Run already exists in server. Run id(s): {8997683}
[2018-04-11 21:36:34] Started run 478 on task 3021, dataset 'sick'.
[2018-04-11 21:36:34] Error in run 471: Run already exists in server. Run id(s): {8997670}
[2018-04-11 21:36:34] Started run 479 on task 3021, dataset 'sick'.
[2018-04-11 21:36:35] Error in run 472: Run already exists in server. Run id(s): {8956295}
[2018-04-11 21:36:35] Started run 480 on task 3021, dataset 'sick'.
[2018-04-11 21:36:37] Error in run 473: Run already exists in server. Run id(s): {8956324}
[2018-04-11 21:36:37] Started run 481 on task 3021, dataset 'sick'.
[2018-04-11 21:36:38] Error in run 474: Run already exists in server. Run id(s): {8956320}
[2018-04-11 21:36:38] Started run 482 on task 3021, dataset 'sick'.
[2018-04-11 21:36:38] Error in run 476: Run already exists in server. Run id(s): {8956317}
[2018-04-11 21:36:38] Started run 483 on task 3021, dataset 'sick'.
[2018-04-11 21:36:38] Error in run 477: Run already exists in server. Run id(s): {8956312}
[2018-04-11 21:36:38] Error in run 475: Run already exists in server. Run id(s): {8997688}
[2018-04-11 21:36:38] Started run 484 on task 3021, dataset 'sick'.
[2018-04-11 21:36:38] Started run 485 on task 3021, dataset 'sick'.
[2018-04-11 21:36:39] Error in run 479: Run already exists in server. Run id(s): {8956274}
[2018-04-11 21:36:39] Started run 486 on task 3021, dataset 'sick'.
[2018-04-11 21:36:39] Error in run 478: Run already exists in server. Run id(s): {8956323}
[2018-04-11 21:36:39] Started run 487 on task 3021, dataset 'sick'.
[2018-04-11 21:36:39] Error in run 480: Run already exists in server. Run id(s): {8956305}
[2018-04-11 21:36:39] Started run 488 on task 3021, dataset 'sick'.
[2018-04-11 21:36:42] Error in run 481: Run already exists in server. Run id(s): {8956341}
[2018-04-11 21:36:42] Started run 489 on task 3021, dataset 'sick'.
[2018-04-11 21:36:42] Error in run 482: Run already exists in server. Run id(s): {8956350}
[2018-04-11 21:36:42] Started run 490 on task 3021, dataset 'sick'.
[2018-04-11 21:36:42] Error in run 483: Run already exists in server. Run id(s): {8956335}
[2018-04-11 21:36:42] Started run 491 on task 3021, dataset 'sick'.
[2018-04-11 21:36:43] Error in run 485: Run already exists in server. Run id(s): {8956325}
[2018-04-11 21:36:43] Started run 492 on task 3021, dataset 'sick'.
[2018-04-11 21:36:43] Error in run 484: Run already exists in server. Run id(s): {8956302}
[2018-04-11 21:36:43] Started run 493 on task 3021, dataset 'sick'.
[2018-04-11 21:36:43] Error in run 486: Run already exists in server. Run id(s): {8956306}
[2018-04-11 21:36:43] Started run 494 on task 3021, dataset 'sick'.
[2018-04-11 21:36:44] Error in run 487: Run already exists in server. Run id(s): {8956308}
[2018-04-11 21:36:44] Error in run 488: Run already exists in server. Run id(s): {8956303}
[2018-04-11 21:36:44] Started run 495 on task 3021, dataset 'sick'.
[2018-04-11 21:36:44] Started run 496 on task 3021, dataset 'sick'.
[2018-04-11 21:36:46] Error in run 489: Run already exists in server. Run id(s): {8956293}
[2018-04-11 21:36:46] Started run 497 on task 3021, dataset 'sick'.
[2018-04-11 21:36:47] Error in run 490: Run already exists in server. Run id(s): {8956342}
[2018-04-11 21:36:47] Started run 498 on task 3021, dataset 'sick'.
[2018-04-11 21:36:47] Error in run 491: Run already exists in server. Run id(s): {8956344}
[2018-04-11 21:36:47] Started run 499 on task 3021, dataset 'sick'.
[2018-04-11 21:36:47] Error in run 492: Run already exists in server. Run id(s): {8956326}
[2018-04-11 21:36:47] Started run 500 on task 3021, dataset 'sick'.
[2018-04-11 21:36:48] Error in run 493: Run already exists in server. Run id(s): {8956322}
[2018-04-11 21:36:48] Started run 501 on task 3021, dataset 'sick'.
[2018-04-11 21:36:48] Error in run 494: Run already exists in server. Run id(s): {8956321}
[2018-04-11 21:36:48] Started run 502 on task 3021, dataset 'sick'.
[2018-04-11 21:36:48] Error in run 495: Run already exists in server. Run id(s): {8956337}
[2018-04-11 21:36:48] Started run 503 on task 3021, dataset 'sick'.
[2018-04-11 21:36:48] Error in run 496: Run already exists in server. Run id(s): {8956336}
[2018-04-11 21:36:48] Started run 504 on task 3021, dataset 'sick'.
[2018-04-11 21:36:51] Error in run 497: Run already exists in server. Run id(s): {8956331}
[2018-04-11 21:36:51] Started run 505 on task 3021, dataset 'sick'.
[2018-04-11 21:36:51] Error in run 498: Run already exists in server. Run id(s): {8956283}
[2018-04-11 21:36:51] Started run 506 on task 3021, dataset 'sick'.
[2018-04-11 21:36:52] Error in run 499: Run already exists in server. Run id(s): {8956316}
[2018-04-11 21:36:52] Started run 507 on task 3021, dataset 'sick'.
[2018-04-11 21:36:52] Error in run 500: Run already exists in server. Run id(s): {8956284}
[2018-04-11 21:36:52] Started run 508 on task 3021, dataset 'sick'.
[2018-04-11 21:36:52] Error in run 501: Run already exists in server. Run id(s): {8956329}
[2018-04-11 21:36:52] Started run 509 on task 3021, dataset 'sick'.
[2018-04-11 21:36:52] Error in run 502: Run already exists in server. Run id(s): {8956330}
[2018-04-11 21:36:52] Started run 510 on task 3021, dataset 'sick'.
[2018-04-11 21:36:53] Error in run 503: Run already exists in server. Run id(s): {8956315}
[2018-04-11 21:36:53] Started run 511 on task 3021, dataset 'sick'.
[2018-04-11 21:36:53] Error in run 504: Run already exists in server. Run id(s): {8956288}
[2018-04-11 21:36:53] Started run 512 on task 3021, dataset 'sick'.
[2018-04-11 21:36:55] Error in run 505: Run already exists in server. Run id(s): {8956332}
[2018-04-11 21:36:55] Started run 513 on task 3021, dataset 'sick'.
[2018-04-11 21:36:56] Error in run 506: Run already exists in server. Run id(s): {8956311}
[2018-04-11 21:36:56] Started run 514 on task 3021, dataset 'sick'.
[2018-04-11 21:36:56] Error in run 507: Run already exists in server. Run id(s): {8997726}
[2018-04-11 21:36:56] Started run 515 on task 3021, dataset 'sick'.
[2018-04-11 21:36:56] Error in run 508: Run already exists in server. Run id(s): {8956328}
[2018-04-11 21:36:56] Started run 516 on task 3021, dataset 'sick'.
[2018-04-11 21:36:56] Error in run 509: Run already exists in server. Run id(s): {8956340}
[2018-04-11 21:36:56] Started run 517 on task 3021, dataset 'sick'.
[2018-04-11 21:36:57] Error in run 510: Run already exists in server. Run id(s): {8956348}
[2018-04-11 21:36:57] Started run 518 on task 3021, dataset 'sick'.
[2018-04-11 21:36:57] Error in run 511: Run already exists in server. Run id(s): {8956357}
[2018-04-11 21:36:57] Started run 519 on task 3021, dataset 'sick'.
[2018-04-11 21:36:57] Error in run 512: Run already exists in server. Run id(s): {8956355}
[2018-04-11 21:36:57] Started run 520 on task 3021, dataset 'sick'.
[2018-04-11 21:37:00] Error in run 513: Run already exists in server. Run id(s): {8956349}
[2018-04-11 21:37:00] Started run 521 on task 3021, dataset 'sick'.
[2018-04-11 21:37:00] Error in run 514: Run already exists in server. Run id(s): {8956351}
[2018-04-11 21:37:00] Started run 522 on task 3021, dataset 'sick'.
[2018-04-11 21:37:01] Error in run 515: Run already exists in server. Run id(s): {8997728}
[2018-04-11 21:37:01] Started run 523 on task 3021, dataset 'sick'.
[2018-04-11 21:37:01] Error in run 516: Run already exists in server. Run id(s): {8956347}
[2018-04-11 21:37:01] Started run 524 on task 3021, dataset 'sick'.
[2018-04-11 21:37:01] Error in run 517: Run already exists in server. Run id(s): {8956319}
[2018-04-11 21:37:01] Started run 525 on task 3021, dataset 'sick'.
[2018-04-11 21:37:01] Error in run 518: Run already exists in server. Run id(s): {8956314}
[2018-04-11 21:37:01] Started run 526 on task 3021, dataset 'sick'.
[2018-04-11 21:37:02] Error in run 519: Run already exists in server. Run id(s): {8956343}
[2018-04-11 21:37:02] Started run 527 on task 3021, dataset 'sick'.
[2018-04-11 21:37:02] Error in run 520: Run already exists in server. Run id(s): {8956363}
[2018-04-11 21:37:02] Started run 528 on task 3021, dataset 'sick'.
[2018-04-11 21:37:04] Error in run 521: Run already exists in server. Run id(s): {8956313}
[2018-04-11 21:37:04] Started run 529 on task 3021, dataset 'sick'.
[2018-04-11 21:37:05] Error in run 522: Run already exists in server. Run id(s): {8956364}
[2018-04-11 21:37:05] Started run 530 on task 3021, dataset 'sick'.
[2018-04-11 21:37:05] Error in run 523: Run already exists in server. Run id(s): {8956338}
[2018-04-11 21:37:05] Started run 531 on task 3021, dataset 'sick'.
[2018-04-11 21:37:05] Error in run 524: Run already exists in server. Run id(s): {8956359}
[2018-04-11 21:37:05] Started run 532 on task 3021, dataset 'sick'.
[2018-04-11 21:37:05] Error in run 525: Run already exists in server. Run id(s): {8956365, 9030495}
[2018-04-11 21:37:05] Started run 533 on task 3021, dataset 'sick'.
[2018-04-11 21:37:06] Error in run 526: Run already exists in server. Run id(s): {8956367}
[2018-04-11 21:37:06] Started run 534 on task 3021, dataset 'sick'.
[2018-04-11 21:37:06] Error in run 527: Run already exists in server. Run id(s): {8956346}
[2018-04-11 21:37:06] Started run 535 on task 3021, dataset 'sick'.
[2018-04-11 21:37:06] Error in run 528: Run already exists in server. Run id(s): {8956370}
[2018-04-11 21:37:06] Started run 536 on task 3021, dataset 'sick'.
[2018-04-11 21:37:09] Error in run 529: Run already exists in server. Run id(s): {8956358}
[2018-04-11 21:37:09] Error in run 530: Run already exists in server. Run id(s): {8956372}
[2018-04-11 21:37:09] Started run 537 on task 3021, dataset 'sick'.
[2018-04-11 21:37:09] Started run 538 on task 3021, dataset 'sick'.
[2018-04-11 21:37:10] Error in run 532: Run already exists in server. Run id(s): {8956373}
[2018-04-11 21:37:10] Started run 539 on task 3021, dataset 'sick'.
[2018-04-11 21:37:10] Error in run 531: Run already exists in server. Run id(s): {8956361}
[2018-04-11 21:37:10] Started run 540 on task 3021, dataset 'sick'.
[2018-04-11 21:37:10] Error in run 533: Run already exists in server. Run id(s): {8956369}
[2018-04-11 21:37:10] Started run 541 on task 3021, dataset 'sick'.
[2018-04-11 21:37:11] Error in run 534: Run already exists in server. Run id(s): {8956360}
[2018-04-11 21:37:11] Error in run 535: Run already exists in server. Run id(s): {8956380}
[2018-04-11 21:37:11] Started run 542 on task 3021, dataset 'sick'.
[2018-04-11 21:37:11] Started run 543 on task 3021, dataset 'sick'.
[2018-04-11 21:37:11] Error in run 536: Run already exists in server. Run id(s): {8956366}
[2018-04-11 21:37:11] Started run 544 on task 3021, dataset 'sick'.
[2018-04-11 21:37:14] Error in run 538: Run already exists in server. Run id(s): {8956377}
[2018-04-11 21:37:14] Started run 545 on task 3021, dataset 'sick'.
[2018-04-11 21:37:14] Error in run 537: Run already exists in server. Run id(s): {8956387}
[2018-04-11 21:37:14] Started run 546 on task 3021, dataset 'sick'.
[2018-04-11 21:37:14] Error in run 539: Run already exists in server. Run id(s): {8956384}
[2018-04-11 21:37:14] Started run 547 on task 3021, dataset 'sick'.
[2018-04-11 21:37:15] Error in run 540: Run already exists in server. Run id(s): {8956374}
[2018-04-11 21:37:15] Error in run 541: Run already exists in server. Run id(s): {8956381}
[2018-04-11 21:37:15] Started run 548 on task 3021, dataset 'sick'.
[2018-04-11 21:37:15] Started run 549 on task 3021, dataset 'sick'.
[2018-04-11 21:37:16] Error in run 542: Run already exists in server. Run id(s): {8956352}
[2018-04-11 21:37:16] Started run 550 on task 3021, dataset 'sick'.
[2018-04-11 21:37:16] Error in run 544: Run already exists in server. Run id(s): {8956385}
[2018-04-11 21:37:16] Started run 551 on task 3021, dataset 'sick'.
[2018-04-11 21:37:16] Error in run 543: Run already exists in server. Run id(s): {8956356}
[2018-04-11 21:37:16] Started run 552 on task 3021, dataset 'sick'.
[2018-04-11 21:37:18] Error in run 545: Run already exists in server. Run id(s): {8956401}
[2018-04-11 21:37:18] Started run 553 on task 3021, dataset 'sick'.
[2018-04-11 21:37:19] Error in run 546: Run already exists in server. Run id(s): {8956376}
[2018-04-11 21:37:19] Started run 554 on task 3021, dataset 'sick'.
[2018-04-11 21:37:19] Error in run 547: Run already exists in server. Run id(s): {8956395}
[2018-04-11 21:37:19] Started run 555 on task 3021, dataset 'sick'.
[2018-04-11 21:37:19] Error in run 549: Run already exists in server. Run id(s): {8956394}
[2018-04-11 21:37:19] Started run 556 on task 3021, dataset 'sick'.
[2018-04-11 21:37:19] Error in run 548: Run already exists in server. Run id(s): {8956375}
[2018-04-11 21:37:19] Started run 557 on task 3021, dataset 'sick'.
[2018-04-11 21:37:20] Error in run 552: Run already exists in server. Run id(s): {8956404}
[2018-04-11 21:37:20] Error in run 550: Run already exists in server. Run id(s): {8956399}
[2018-04-11 21:37:20] Error in run 551: Run already exists in server. Run id(s): {8956414}
[2018-04-11 21:37:20] Started run 558 on task 3021, dataset 'sick'.
[2018-04-11 21:37:20] Started run 559 on task 3021, dataset 'sick'.
[2018-04-11 21:37:20] Started run 560 on task 3021, dataset 'sick'.
[2018-04-11 21:37:23] Error in run 553: Run already exists in server. Run id(s): {8956389}
[2018-04-11 21:37:23] Started run 561 on task 3021, dataset 'sick'.
[2018-04-11 21:37:23] Error in run 554: Run already exists in server. Run id(s): {8956371}
[2018-04-11 21:37:23] Started run 562 on task 3021, dataset 'sick'.
[2018-04-11 21:37:23] Error in run 555: Run already exists in server. Run id(s): {8956386}
[2018-04-11 21:37:23] Started run 563 on task 3021, dataset 'sick'.
[2018-04-11 21:37:24] Error in run 556: Run already exists in server. Run id(s): {8956410}
[2018-04-11 21:37:24] Started run 564 on task 3021, dataset 'sick'.
[2018-04-11 21:37:24] Error in run 557: Run already exists in server. Run id(s): {8956415}
[2018-04-11 21:37:24] Started run 565 on task 3021, dataset 'sick'.
[2018-04-11 21:37:25] Error in run 558: Run already exists in server. Run id(s): {8956411}
[2018-04-11 21:37:25] Error in run 559: Run already exists in server. Run id(s): {8956408}
[2018-04-11 21:37:25] Started run 566 on task 3021, dataset 'sick'.
[2018-04-11 21:37:25] Started run 567 on task 3021, dataset 'sick'.
[2018-04-11 21:37:25] Error in run 560: Run already exists in server. Run id(s): {8956409}
[2018-04-11 21:37:25] Started run 568 on task 3021, dataset 'sick'.
[2018-04-11 21:37:27] Error in run 561: Run already exists in server. Run id(s): {8997766}
[2018-04-11 21:37:27] Started run 569 on task 3021, dataset 'sick'.
[2018-04-11 21:37:28] Error in run 562: Run already exists in server. Run id(s): {8956397}
[2018-04-11 21:37:28] Started run 570 on task 3021, dataset 'sick'.
[2018-04-11 21:37:28] Error in run 563: Run already exists in server. Run id(s): {8997757}
[2018-04-11 21:37:28] Started run 571 on task 3021, dataset 'sick'.
[2018-04-11 21:37:28] Error in run 564: Run already exists in server. Run id(s): {8956407}
[2018-04-11 21:37:28] Started run 572 on task 3021, dataset 'sick'.
[2018-04-11 21:37:28] Error in run 565: Run already exists in server. Run id(s): {8997763}
[2018-04-11 21:37:28] Started run 573 on task 3021, dataset 'sick'.
[2018-04-11 21:37:29] Error in run 566: Run already exists in server. Run id(s): {8997767}
[2018-04-11 21:37:29] Started run 574 on task 3021, dataset 'sick'.
[2018-04-11 21:37:29] Error in run 567: Run already exists in server. Run id(s): {8997751}
[2018-04-11 21:37:29] Started run 575 on task 3021, dataset 'sick'.
[2018-04-11 21:37:29] Error in run 568: Run already exists in server. Run id(s): {8997746}
[2018-04-11 21:37:29] Started run 576 on task 3021, dataset 'sick'.
[2018-04-11 21:37:32] Error in run 569: Run already exists in server. Run id(s): {8997790}
[2018-04-11 21:37:32] Started run 577 on task 3021, dataset 'sick'.
[2018-04-11 21:37:32] Error in run 571: Run already exists in server. Run id(s): {8997784}
[2018-04-11 21:37:32] Error in run 570: Run already exists in server. Run id(s): {8956426}
[2018-04-11 21:37:32] Started run 578 on task 3021, dataset 'sick'.
[2018-04-11 21:37:32] Started run 579 on task 3021, dataset 'sick'.
[2018-04-11 21:37:33] Error in run 572: Run already exists in server. Run id(s): {8956383}
[2018-04-11 21:37:33] Started run 580 on task 3021, dataset 'sick'.
[2018-04-11 21:37:33] Error in run 573: Run already exists in server. Run id(s): {8956396}
[2018-04-11 21:37:33] Started run 581 on task 3021, dataset 'sick'.
[2018-04-11 21:37:34] Error in run 574: Run already exists in server. Run id(s): {8997801}
[2018-04-11 21:37:34] Started run 582 on task 3021, dataset 'sick'.
[2018-04-11 21:37:34] Error in run 575: Run already exists in server. Run id(s): {8997779}
[2018-04-11 21:37:34] Started run 583 on task 3021, dataset 'sick'.
[2018-04-11 21:37:34] Error in run 576: Run already exists in server. Run id(s): {8997800}
[2018-04-11 21:37:34] Started run 584 on task 3021, dataset 'sick'.
[2018-04-11 21:37:36] Error in run 577: Run already exists in server. Run id(s): {8997802}
[2018-04-11 21:37:36] Started run 585 on task 3021, dataset 'sick'.
[2018-04-11 21:37:37] Error in run 578: Run already exists in server. Run id(s): {8997821}
[2018-04-11 21:37:37] Started run 586 on task 3021, dataset 'sick'.
[2018-04-11 21:37:37] Error in run 579: Run already exists in server. Run id(s): {8956436}
[2018-04-11 21:37:37] Started run 587 on task 3021, dataset 'sick'.
[2018-04-11 21:37:37] Error in run 580: Run already exists in server. Run id(s): {8956428}
[2018-04-11 21:37:37] Started run 588 on task 3021, dataset 'sick'.
[2018-04-11 21:37:38] Error in run 581: Run already exists in server. Run id(s): {8956434}
[2018-04-11 21:37:38] Started run 589 on task 3021, dataset 'sick'.
[2018-04-11 21:37:38] Error in run 582: Run already exists in server. Run id(s): {8956431}
[2018-04-11 21:37:38] Error in run 583: Run already exists in server. Run id(s): {8956440}
[2018-04-11 21:37:38] Started run 590 on task 3021, dataset 'sick'.
[2018-04-11 21:37:38] Started run 591 on task 3021, dataset 'sick'.
[2018-04-11 21:37:38] Error in run 584: Run already exists in server. Run id(s): {8956400}
[2018-04-11 21:37:38] Started run 592 on task 3021, dataset 'sick'.
[2018-04-11 21:37:41] Error in run 585: Run already exists in server. Run id(s): {8956427}
[2018-04-11 21:37:41] Started run 593 on task 3021, dataset 'sick'.
[2018-04-11 21:37:42] Error in run 587: Run already exists in server. Run id(s): {8956432}
[2018-04-11 21:37:42] Started run 594 on task 3021, dataset 'sick'.
[2018-04-11 21:37:42] Error in run 586: Run already exists in server. Run id(s): {8956416}
[2018-04-11 21:37:42] Started run 595 on task 3021, dataset 'sick'.
[2018-04-11 21:37:42] Error in run 588: Run already exists in server. Run id(s): {8956439}
[2018-04-11 21:37:42] Started run 596 on task 3021, dataset 'sick'.
[2018-04-11 21:37:42] Error in run 589: Run already exists in server. Run id(s): {8956419}
[2018-04-11 21:37:42] Started run 597 on task 3021, dataset 'sick'.
[2018-04-11 21:37:43] Error in run 591: Run already exists in server. Run id(s): {8956443}
[2018-04-11 21:37:43] Error in run 590: Run already exists in server. Run id(s): {8956413}
[2018-04-11 21:37:43] Error in run 592: Run already exists in server. Run id(s): {8956435}
[2018-04-11 21:37:43] Started run 598 on task 3021, dataset 'sick'.
[2018-04-11 21:37:43] Started run 600 on task 3021, dataset 'sick'.
[2018-04-11 21:37:43] Started run 599 on task 3021, dataset 'sick'.
[2018-04-11 21:37:45] Error in run 593: Run already exists in server. Run id(s): {8956418}
[2018-04-11 21:37:45] Started run 601 on task 3021, dataset 'sick'.
[2018-04-11 21:37:46] Error in run 594: Run already exists in server. Run id(s): {8956392}
[2018-04-11 21:37:46] Started run 602 on task 3021, dataset 'sick'.
[2018-04-11 21:37:46] Error in run 595: Run already exists in server. Run id(s): {8956421}
[2018-04-11 21:37:46] Started run 603 on task 3021, dataset 'sick'.
[2018-04-11 21:37:46] Error in run 596: Run already exists in server. Run id(s): {8956393}
[2018-04-11 21:37:46] Started run 604 on task 3021, dataset 'sick'.
[2018-04-11 21:37:47] Error in run 597: Run already exists in server. Run id(s): {8956448}
[2018-04-11 21:37:47] Started run 605 on task 3021, dataset 'sick'.
[2018-04-11 21:37:48] Error in run 600: Run already exists in server. Run id(s): {8956420}
[2018-04-11 21:37:48] Error in run 598: Run already exists in server. Run id(s): {8956425}
[2018-04-11 21:37:48] Error in run 599: Run already exists in server. Run id(s): {8956390}
[2018-04-11 21:37:48] Started run 606 on task 3021, dataset 'sick'.
[2018-04-11 21:37:48] Started run 607 on task 3021, dataset 'sick'.
[2018-04-11 21:37:48] Started run 608 on task 3021, dataset 'sick'.
[2018-04-11 21:37:50] Error in run 601: Run already exists in server. Run id(s): {8956455}
[2018-04-11 21:37:50] Started run 609 on task 3021, dataset 'sick'.
[2018-04-11 21:37:50] Error in run 602: Run already exists in server. Run id(s): {8956444}
[2018-04-11 21:37:50] Started run 610 on task 3021, dataset 'sick'.
[2018-04-11 21:37:51] Error in run 603: Run already exists in server. Run id(s): {8956458}
[2018-04-11 21:37:51] Started run 611 on task 3021, dataset 'sick'.
[2018-04-11 21:37:51] Error in run 604: Run already exists in server. Run id(s): {8956441}
[2018-04-11 21:37:51] Started run 612 on task 3021, dataset 'sick'.
[2018-04-11 21:37:52] Error in run 605: Run already exists in server. Run id(s): {8956422}
[2018-04-11 21:37:52] Started run 613 on task 3021, dataset 'sick'.
[2018-04-11 21:37:53] Error in run 606: Run already exists in server. Run id(s): {8956453}
[2018-04-11 21:37:53] Error in run 607: Run already exists in server. Run id(s): {8956461}
[2018-04-11 21:37:53] Error in run 608: Run already exists in server. Run id(s): {8956445}
[2018-04-11 21:37:53] Started run 614 on task 3021, dataset 'sick'.
[2018-04-11 21:37:53] Started run 615 on task 3021, dataset 'sick'.
[2018-04-11 21:37:53] Started run 616 on task 3021, dataset 'sick'.
[2018-04-11 21:37:55] Error in run 610: Run already exists in server. Run id(s): {8956429}
[2018-04-11 21:37:55] Error in run 609: Run already exists in server. Run id(s): {8956447}
[2018-04-11 21:37:55] Started run 617 on task 3021, dataset 'sick'.
[2018-04-11 21:37:55] Started run 618 on task 3021, dataset 'sick'.
[2018-04-11 21:37:56] Error in run 612: Run already exists in server. Run id(s): {8956430}
[2018-04-11 21:37:56] Error in run 611: Run already exists in server. Run id(s): {8956459}
[2018-04-11 21:37:56] Started run 619 on task 3021, dataset 'sick'.
[2018-04-11 21:37:56] Started run 620 on task 3021, dataset 'sick'.
[2018-04-11 21:37:57] Error in run 613: Run already exists in server. Run id(s): {8956452}
[2018-04-11 21:37:57] Started run 621 on task 3021, dataset 'sick'.
[2018-04-11 21:37:57] Error in run 615: Run already exists in server. Run id(s): {8956464}
[2018-04-11 21:37:57] Error in run 616: Run already exists in server. Run id(s): {8956457}
[2018-04-11 21:37:57] Error in run 614: Run already exists in server. Run id(s): {8956466}
[2018-04-11 21:37:57] Started run 622 on task 3021, dataset 'sick'.
[2018-04-11 21:37:57] Started run 624 on task 3021, dataset 'sick'.
[2018-04-11 21:37:57] Started run 623 on task 3021, dataset 'sick'.
[2018-04-11 21:38:00] Error in run 618: Run already exists in server. Run id(s): {8956438}
[2018-04-11 21:38:00] Error in run 617: Run already exists in server. Run id(s): {8956460}
[2018-04-11 21:38:00] Started run 625 on task 3021, dataset 'sick'.
[2018-04-11 21:38:00] Started run 626 on task 3021, dataset 'sick'.
[2018-04-11 21:38:01] Error in run 620: Run already exists in server. Run id(s): {8956442}
[2018-04-11 21:38:01] Error in run 619: Run already exists in server. Run id(s): {8956469}
[2018-04-11 21:38:01] Started run 627 on task 3021, dataset 'sick'.
[2018-04-11 21:38:01] Started run 628 on task 3021, dataset 'sick'.
[2018-04-11 21:38:01] Error in run 621: Run already exists in server. Run id(s): {8956467}
[2018-04-11 21:38:01] Started run 629 on task 3021, dataset 'sick'.
[2018-04-11 21:38:02] Error in run 622: Run already exists in server. Run id(s): {8956456}
[2018-04-11 21:38:02] Started run 630 on task 3021, dataset 'sick'.
[2018-04-11 21:38:02] Error in run 624: Run already exists in server. Run id(s): {8956471}
[2018-04-11 21:38:02] Error in run 623: Run already exists in server. Run id(s): {8956406}
[2018-04-11 21:38:02] Started run 631 on task 3021, dataset 'sick'.
[2018-04-11 21:38:02] Started run 632 on task 3021, dataset 'sick'.
[2018-04-11 21:38:04] Error in run 625: Run already exists in server. Run id(s): {8956463}
[2018-04-11 21:38:04] Started run 633 on task 3021, dataset 'sick'.
[2018-04-11 21:38:04] Error in run 626: Run already exists in server. Run id(s): {8956465}
[2018-04-11 21:38:04] Started run 634 on task 3021, dataset 'sick'.
[2018-04-11 21:38:05] Error in run 627: Run already exists in server. Run id(s): {8956481}
[2018-04-11 21:38:05] Started run 635 on task 3021, dataset 'sick'.
[2018-04-11 21:38:05] Error in run 628: Run already exists in server. Run id(s): {8956462}
[2018-04-11 21:38:05] Started run 636 on task 3021, dataset 'sick'.
[2018-04-11 21:38:06] Error in run 629: Run already exists in server. Run id(s): {8956450}
[2018-04-11 21:38:06] Started run 637 on task 3021, dataset 'sick'.
[2018-04-11 21:38:07] Error in run 630: Run already exists in server. Run id(s): {8956472}
[2018-04-11 21:38:07] Started run 638 on task 3021, dataset 'sick'.
[2018-04-11 21:38:07] Error in run 632: Run already exists in server. Run id(s): {8956493}
[2018-04-11 21:38:07] Error in run 631: Run already exists in server. Run id(s): {8956424}
[2018-04-11 21:38:07] Started run 639 on task 3021, dataset 'sick'.
[2018-04-11 21:38:07] Started run 640 on task 3021, dataset 'sick'.
[2018-04-11 21:38:09] Error in run 634: Run already exists in server. Run id(s): {8956477}
[2018-04-11 21:38:09] Started run 641 on task 3021, dataset 'sick'.
[2018-04-11 21:38:09] Error in run 633: Run already exists in server. Run id(s): {8956454}
[2018-04-11 21:38:09] Started run 642 on task 3021, dataset 'sick'.
[2018-04-11 21:38:10] Error in run 635: Run already exists in server. Run id(s): {8956482}
[2018-04-11 21:38:10] Started run 643 on task 3021, dataset 'sick'.
[2018-04-11 21:38:10] Error in run 636: Run already exists in server. Run id(s): {8956479}
[2018-04-11 21:38:10] Started run 644 on task 3021, dataset 'sick'.
[2018-04-11 21:38:11] Error in run 637: Run already exists in server. Run id(s): {8956492}
[2018-04-11 21:38:11] Started run 645 on task 3021, dataset 'sick'.
[2018-04-11 21:38:11] Error in run 638: Run already exists in server. Run id(s): {8956476}
[2018-04-11 21:38:11] Started run 646 on task 3021, dataset 'sick'.
[2018-04-11 21:38:11] Error in run 640: Run already exists in server. Run id(s): {8956480}
[2018-04-11 21:38:11] Error in run 639: Run already exists in server. Run id(s): {8956485}
[2018-04-11 21:38:11] Started run 647 on task 3021, dataset 'sick'.
[2018-04-11 21:38:11] Started run 648 on task 3021, dataset 'sick'.
[2018-04-11 21:38:13] Error in run 641: Run already exists in server. Run id(s): {8956489}
[2018-04-11 21:38:13] Started run 649 on task 3021, dataset 'sick'.
[2018-04-11 21:38:13] Error in run 642: Run already exists in server. Run id(s): {8956483}
[2018-04-11 21:38:13] Started run 650 on task 3021, dataset 'sick'.
[2018-04-11 21:38:14] Error in run 643: Run already exists in server. Run id(s): {8956484}
[2018-04-11 21:38:14] Started run 651 on task 3021, dataset 'sick'.
[2018-04-11 21:38:14] Error in run 644: Run already exists in server. Run id(s): {8956497}
[2018-04-11 21:38:14] Started run 652 on task 3021, dataset 'sick'.
[2018-04-11 21:38:15] Error in run 645: Run already exists in server. Run id(s): {8956474}
[2018-04-11 21:38:15] Started run 653 on task 3021, dataset 'sick'.
[2018-04-11 21:38:16] Error in run 646: Run already exists in server. Run id(s): {8956507}
[2018-04-11 21:38:16] Started run 654 on task 3021, dataset 'sick'.
[2018-04-11 21:38:16] Error in run 647: Run already exists in server. Run id(s): {8956473}
[2018-04-11 21:38:16] Error in run 648: Run already exists in server. Run id(s): {8956486}
[2018-04-11 21:38:16] Started run 655 on task 3021, dataset 'sick'.
[2018-04-11 21:38:16] Started run 656 on task 3021, dataset 'sick'.
[2018-04-11 21:38:18] Error in run 649: Run already exists in server. Run id(s): {8956490}
[2018-04-11 21:38:18] Error in run 650: Run already exists in server. Run id(s): {8956470}
[2018-04-11 21:38:18] Started run 657 on task 3021, dataset 'sick'.
[2018-04-11 21:38:18] Started run 658 on task 3021, dataset 'sick'.
[2018-04-11 21:38:19] Error in run 651: Run already exists in server. Run id(s): {8956505}
[2018-04-11 21:38:19] Started run 659 on task 3021, dataset 'sick'.
[2018-04-11 21:38:19] Error in run 652: Run already exists in server. Run id(s): {8956503}
[2018-04-11 21:38:19] Started run 660 on task 3021, dataset 'sick'.
[2018-04-11 21:38:20] Error in run 653: Run already exists in server. Run id(s): {8956509}
[2018-04-11 21:38:20] Started run 661 on task 3021, dataset 'sick'.
[2018-04-11 21:38:20] Error in run 654: Run already exists in server. Run id(s): {8956478}
[2018-04-11 21:38:20] Started run 662 on task 3021, dataset 'sick'.
[2018-04-11 21:38:20] Error in run 656: Run already exists in server. Run id(s): {8956499}
[2018-04-11 21:38:20] Error in run 655: Run already exists in server. Run id(s): {8956494}
[2018-04-11 21:38:20] Started run 663 on task 3021, dataset 'sick'.
[2018-04-11 21:38:20] Started run 664 on task 3021, dataset 'sick'.
[2018-04-11 21:38:23] Error in run 658: Run already exists in server. Run id(s): {8956512}
[2018-04-11 21:38:23] Started run 665 on task 3021, dataset 'sick'.
[2018-04-11 21:38:23] Error in run 657: Run already exists in server. Run id(s): {8956501}
[2018-04-11 21:38:23] Started run 666 on task 3021, dataset 'sick'.
[2018-04-11 21:38:23] Error in run 659: Run already exists in server. Run id(s): {8956517}
[2018-04-11 21:38:23] Started run 667 on task 3021, dataset 'sick'.
[2018-04-11 21:38:23] Error in run 660: Run already exists in server. Run id(s): {8956519}
[2018-04-11 21:38:23] Started run 668 on task 3021, dataset 'sick'.
[2018-04-11 21:38:24] Error in run 661: Run already exists in server. Run id(s): {8956508}
[2018-04-11 21:38:24] Started run 669 on task 3021, dataset 'sick'.
[2018-04-11 21:38:25] Error in run 662: Run already exists in server. Run id(s): {8956504}
[2018-04-11 21:38:25] Error in run 663: Run already exists in server. Run id(s): {8956513}
[2018-04-11 21:38:25] Started run 670 on task 3021, dataset 'sick'.
[2018-04-11 21:38:25] Error in run 664: Run already exists in server. Run id(s): {8956496}
[2018-04-11 21:38:25] Started run 671 on task 3021, dataset 'sick'.
[2018-04-11 21:38:25] Started run 672 on task 3021, dataset 'sick'.
[2018-04-11 21:38:27] Error in run 665: Run already exists in server. Run id(s): {8956518}
[2018-04-11 21:38:27] Error in run 666: Run already exists in server. Run id(s): {8956506}
[2018-04-11 21:38:27] Started run 673 on task 3021, dataset 'sick'.
[2018-04-11 21:38:27] Started run 674 on task 3021, dataset 'sick'.
[2018-04-11 21:38:28] Error in run 667: Run already exists in server. Run id(s): {8956487}
[2018-04-11 21:38:28] Started run 675 on task 3021, dataset 'sick'.
[2018-04-11 21:38:28] Error in run 668: Run already exists in server. Run id(s): {8956511}
[2018-04-11 21:38:28] Started run 676 on task 3021, dataset 'sick'.
[2018-04-11 21:38:29] Error in run 669: Run already exists in server. Run id(s): {8956529}
[2018-04-11 21:38:29] Started run 677 on task 3021, dataset 'sick'.
[2018-04-11 21:38:30] Error in run 671: Run already exists in server. Run id(s): {8956514}
[2018-04-11 21:38:30] Error in run 672: Run already exists in server. Run id(s): {8956522}
[2018-04-11 21:38:30] Error in run 670: Run already exists in server. Run id(s): {8956528}
[2018-04-11 21:38:30] Started run 678 on task 3021, dataset 'sick'.
[2018-04-11 21:38:30] Started run 680 on task 3021, dataset 'sick'.
[2018-04-11 21:38:30] Started run 679 on task 3021, dataset 'sick'.
[2018-04-11 21:38:32] Error in run 674: Run already exists in server. Run id(s): {8997888}
[2018-04-11 21:38:32] Error in run 673: Run already exists in server. Run id(s): {8997898}
[2018-04-11 21:38:32] Error in run 675: Run already exists in server. Run id(s): {8956510}
[2018-04-11 21:38:32] Started run 681 on task 3021, dataset 'sick'.
[2018-04-11 21:38:32] Started run 682 on task 3021, dataset 'sick'.
[2018-04-11 21:38:32] Started run 683 on task 3021, dataset 'sick'.
[2018-04-11 21:38:33] Error in run 676: Run already exists in server. Run id(s): {8997860}
[2018-04-11 21:38:33] Started run 684 on task 3021, dataset 'sick'.
[2018-04-11 21:38:34] Error in run 677: Run already exists in server. Run id(s): {8956525}
[2018-04-11 21:38:34] Started run 685 on task 3021, dataset 'sick'.
[2018-04-11 21:38:34] Error in run 680: Run already exists in server. Run id(s): {8956523}
[2018-04-11 21:38:34] Started run 686 on task 3021, dataset 'sick'.
[2018-04-11 21:38:35] Error in run 679: Run already exists in server. Run id(s): {8956524}
[2018-04-11 21:38:35] Error in run 678: Run already exists in server. Run id(s): {8997874}
[2018-04-11 21:38:35] Started run 687 on task 3021, dataset 'sick'.
[2018-04-11 21:38:35] Started run 688 on task 3021, dataset 'sick'.
[2018-04-11 21:38:37] Error in run 682: Run already exists in server. Run id(s): {8956542}
[2018-04-11 21:38:37] Started run 689 on task 3021, dataset 'sick'.
[2018-04-11 21:38:37] Error in run 681: Run already exists in server. Run id(s): {8956526}
[2018-04-11 21:38:37] Started run 690 on task 3021, dataset 'sick'.
[2018-04-11 21:38:37] Error in run 683: Run already exists in server. Run id(s): {8956498}
[2018-04-11 21:38:37] Started run 691 on task 3021, dataset 'sick'.
[2018-04-11 21:38:37] Error in run 684: Run already exists in server. Run id(s): {8997893}
[2018-04-11 21:38:37] Started run 692 on task 3021, dataset 'sick'.
[2018-04-11 21:38:38] Error in run 685: Run already exists in server. Run id(s): {8997876}
[2018-04-11 21:38:38] Started run 693 on task 3021, dataset 'sick'.
[2018-04-11 21:38:39] Error in run 686: Run already exists in server. Run id(s): {8956535}
[2018-04-11 21:38:39] Started run 694 on task 3021, dataset 'sick'.
[2018-04-11 21:38:39] Error in run 687: Run already exists in server. Run id(s): {8956539}
[2018-04-11 21:38:39] Error in run 688: Run already exists in server. Run id(s): {8997899}
[2018-04-11 21:38:39] Started run 695 on task 3021, dataset 'sick'.
[2018-04-11 21:38:39] Started run 696 on task 3021, dataset 'sick'.
[2018-04-11 21:38:41] Error in run 690: Run already exists in server. Run id(s): {8997877}
[2018-04-11 21:38:41] Started run 697 on task 3021, dataset 'sick'.
[2018-04-11 21:38:41] Error in run 689: Run already exists in server. Run id(s): {8956537}
[2018-04-11 21:38:41] Started run 698 on task 3021, dataset 'sick'.
[2018-04-11 21:38:42] Error in run 692: Run already exists in server. Run id(s): {8997920}
[2018-04-11 21:38:42] Error in run 691: Run already exists in server. Run id(s): {8956552}
[2018-04-11 21:38:42] Started run 699 on task 3021, dataset 'sick'.
[2018-04-11 21:38:42] Started run 700 on task 3021, dataset 'sick'.
[2018-04-11 21:38:43] Error in run 693: Run already exists in server. Run id(s): {8997929}
[2018-04-11 21:38:43] Started run 701 on task 3021, dataset 'sick'.
[2018-04-11 21:38:43] Error in run 694: Run already exists in server. Run id(s): {8956554}
[2018-04-11 21:38:43] Started run 702 on task 3021, dataset 'sick'.
[2018-04-11 21:38:44] Error in run 696: Run already exists in server. Run id(s): {8956515}
[2018-04-11 21:38:44] Error in run 695: Run already exists in server. Run id(s): {8956549}
[2018-04-11 21:38:44] Started run 703 on task 3021, dataset 'sick'.
[2018-04-11 21:38:44] Started run 704 on task 3021, dataset 'sick'.
[2018-04-11 21:38:46] Error in run 698: Run already exists in server. Run id(s): {8956559}
[2018-04-11 21:38:46] Error in run 697: Run already exists in server. Run id(s): {8956547}
[2018-04-11 21:38:46] Started run 705 on task 3021, dataset 'sick'.
[2018-04-11 21:38:46] Started run 706 on task 3021, dataset 'sick'.
[2018-04-11 21:38:46] Error in run 700: Run already exists in server. Run id(s): {8997956}
[2018-04-11 21:38:46] Error in run 699: Run already exists in server. Run id(s): {8956569}
[2018-04-11 21:38:46] Started run 707 on task 3021, dataset 'sick'.
[2018-04-11 21:38:46] Started run 708 on task 3021, dataset 'sick'.
[2018-04-11 21:38:48] Error in run 701: Run already exists in server. Run id(s): {8956536}
[2018-04-11 21:38:48] Error in run 702: Run already exists in server. Run id(s): {8956562}
[2018-04-11 21:38:48] Started run 709 on task 3021, dataset 'sick'.
[2018-04-11 21:38:48] Started run 710 on task 3021, dataset 'sick'.
[2018-04-11 21:38:49] Error in run 703: Run already exists in server. Run id(s): {8956546}
[2018-04-11 21:38:49] Error in run 704: Run already exists in server. Run id(s): {8997936}
[2018-04-11 21:38:49] Started run 711 on task 3021, dataset 'sick'.
[2018-04-11 21:38:49] Started run 712 on task 3021, dataset 'sick'.
[2018-04-11 21:38:50] Error in run 705: Run already exists in server. Run id(s): {8956557}
[2018-04-11 21:38:50] Error in run 706: Run already exists in server. Run id(s): {9030828, 8956567}
[2018-04-11 21:38:50] Started run 713 on task 3021, dataset 'sick'.
[2018-04-11 21:38:50] Started run 714 on task 3021, dataset 'sick'.
[2018-04-11 21:38:51] Error in run 707: Run already exists in server. Run id(s): {8956579}
[2018-04-11 21:38:51] Error in run 708: Run already exists in server. Run id(s): {8956566}
[2018-04-11 21:38:51] Started run 715 on task 3021, dataset 'sick'.
[2018-04-11 21:38:51] Started run 716 on task 3021, dataset 'sick'.
[2018-04-11 21:38:53] Error in run 709: Run already exists in server. Run id(s): {8956560}
[2018-04-11 21:38:53] Started run 717 on task 3021, dataset 'sick'.
[2018-04-11 21:38:53] Error in run 710: Run already exists in server. Run id(s): {8956556}
[2018-04-11 21:38:53] Started run 718 on task 3021, dataset 'sick'.
[2018-04-11 21:38:54] Error in run 711: Run already exists in server. Run id(s): {8956531}
[2018-04-11 21:38:54] Started run 719 on task 3021, dataset 'sick'.
[2018-04-11 21:38:54] Error in run 712: Run already exists in server. Run id(s): {8956543}
[2018-04-11 21:38:54] Started run 720 on task 3021, dataset 'sick'.
[2018-04-11 21:38:55] Error in run 713: Run already exists in server. Run id(s): {8956541}
[2018-04-11 21:38:55] Started run 721 on task 3021, dataset 'sick'.
[2018-04-11 21:38:55] Error in run 714: Run already exists in server. Run id(s): {8956577}
[2018-04-11 21:38:55] Started run 722 on task 3021, dataset 'sick'.
[2018-04-11 21:38:56] Error in run 715: Run already exists in server. Run id(s): {8956561}
[2018-04-11 21:38:56] Error in run 716: Run already exists in server. Run id(s): {8956533}
[2018-04-11 21:38:56] Started run 723 on task 3021, dataset 'sick'.
[2018-04-11 21:38:56] Started run 724 on task 3021, dataset 'sick'.
[2018-04-11 21:38:57] Error in run 717: Run already exists in server. Run id(s): {8956575}
[2018-04-11 21:38:57] Started run 725 on task 3021, dataset 'sick'.
[2018-04-11 21:38:57] Error in run 718: Run already exists in server. Run id(s): {8956568}
[2018-04-11 21:38:57] Started run 726 on task 3021, dataset 'sick'.
[2018-04-11 21:38:58] Error in run 719: Run already exists in server. Run id(s): {8956580}
[2018-04-11 21:38:58] Started run 727 on task 3021, dataset 'sick'.
[2018-04-11 21:38:58] Error in run 720: Run already exists in server. Run id(s): {8956540}
[2018-04-11 21:38:58] Started run 728 on task 3021, dataset 'sick'.
[2018-04-11 21:39:00] Error in run 722: Run already exists in server. Run id(s): {8956576}
[2018-04-11 21:39:00] Error in run 721: Run already exists in server. Run id(s): {8956553}
[2018-04-11 21:39:00] Started run 729 on task 3021, dataset 'sick'.
[2018-04-11 21:39:00] Started run 730 on task 3021, dataset 'sick'.
[2018-04-11 21:39:01] Error in run 724: Run already exists in server. Run id(s): {8956563}
[2018-04-11 21:39:01] Error in run 723: Run already exists in server. Run id(s): {8956587}
[2018-04-11 21:39:01] Started run 731 on task 3021, dataset 'sick'.
[2018-04-11 21:39:01] Started run 732 on task 3021, dataset 'sick'.
[2018-04-11 21:39:02] Error in run 725: Run already exists in server. Run id(s): {8956594}
[2018-04-11 21:39:02] Started run 733 on task 3021, dataset 'sick'.
[2018-04-11 21:39:02] Error in run 726: Run already exists in server. Run id(s): {8956570}
[2018-04-11 21:39:02] Started run 734 on task 3021, dataset 'sick'.
[2018-04-11 21:39:03] Error in run 727: Run already exists in server. Run id(s): {8956581}
[2018-04-11 21:39:03] Started run 735 on task 3021, dataset 'sick'.
[2018-04-11 21:39:03] Error in run 728: Run already exists in server. Run id(s): {8956555}
[2018-04-11 21:39:03] Started run 736 on task 3021, dataset 'sick'.
[2018-04-11 21:39:05] Error in run 729: Run already exists in server. Run id(s): {8956574}
[2018-04-11 21:39:05] Error in run 730: Run already exists in server. Run id(s): {8956530}
[2018-04-11 21:39:05] Started run 737 on task 3021, dataset 'sick'.
[2018-04-11 21:39:05] Started run 738 on task 3021, dataset 'sick'.
[2018-04-11 21:39:05] Error in run 732: Run already exists in server. Run id(s): {8956564}
[2018-04-11 21:39:05] Started run 739 on task 3021, dataset 'sick'.
[2018-04-11 21:39:05] Error in run 731: Run already exists in server. Run id(s): {8956571}
[2018-04-11 21:39:05] Started run 740 on task 3021, dataset 'sick'.
[2018-04-11 21:39:06] Error in run 733: Run already exists in server. Run id(s): {8956589}
[2018-04-11 21:39:06] Started run 741 on task 3021, dataset 'sick'.
[2018-04-11 21:39:06] Error in run 734: Run already exists in server. Run id(s): {8956572}
[2018-04-11 21:39:06] Started run 742 on task 3021, dataset 'sick'.
[2018-04-11 21:39:07] Error in run 735: Run already exists in server. Run id(s): {8956545}
[2018-04-11 21:39:07] Started run 743 on task 3021, dataset 'sick'.
[2018-04-11 21:39:08] Error in run 736: Run already exists in server. Run id(s): {8956593}
[2018-04-11 21:39:08] Started run 744 on task 3021, dataset 'sick'.
[2018-04-11 21:39:09] Error in run 737: Run already exists in server. Run id(s): {8956584}
[2018-04-11 21:39:09] Started run 745 on task 3021, dataset 'sick'.
[2018-04-11 21:39:09] Error in run 738: Run already exists in server. Run id(s): {8956605}
[2018-04-11 21:39:09] Started run 746 on task 3021, dataset 'sick'.
[2018-04-11 21:39:10] Error in run 739: Run already exists in server. Run id(s): {8956551}
[2018-04-11 21:39:10] Started run 747 on task 3021, dataset 'sick'.
[2018-04-11 21:39:10] Error in run 740: Run already exists in server. Run id(s): {8956558}
[2018-04-11 21:39:10] Started run 748 on task 3021, dataset 'sick'.
[2018-04-11 21:39:11] Error in run 741: Run already exists in server. Run id(s): {8956582}
[2018-04-11 21:39:11] Started run 749 on task 3021, dataset 'sick'.
[2018-04-11 21:39:11] Error in run 742: Run already exists in server. Run id(s): {8956598}
[2018-04-11 21:39:11] Started run 750 on task 3021, dataset 'sick'.
[2018-04-11 21:39:12] Error in run 743: Run already exists in server. Run id(s): {8956586}
[2018-04-11 21:39:12] Started run 751 on task 3021, dataset 'sick'.
[2018-04-11 21:39:12] Error in run 744: Run already exists in server. Run id(s): {8956595}
[2018-04-11 21:39:12] Started run 752 on task 3021, dataset 'sick'.
[2018-04-11 21:39:13] Error in run 745: Run already exists in server. Run id(s): {8956609}
[2018-04-11 21:39:13] Started run 753 on task 3021, dataset 'sick'.
[2018-04-11 21:39:13] Error in run 746: Run already exists in server. Run id(s): {8997977}
[2018-04-11 21:39:13] Started run 754 on task 3021, dataset 'sick'.
[2018-04-11 21:39:14] Error in run 747: Run already exists in server. Run id(s): {8956596}
[2018-04-11 21:39:14] Started run 755 on task 3021, dataset 'sick'.
[2018-04-11 21:39:15] Error in run 748: Run already exists in server. Run id(s): {8956583}
[2018-04-11 21:39:15] Started run 756 on task 3021, dataset 'sick'.
[2018-04-11 21:39:16] Error in run 749: Run already exists in server. Run id(s): {8956606}
[2018-04-11 21:39:16] Error in run 750: Run already exists in server. Run id(s): {8956588}
[2018-04-11 21:39:16] Started run 757 on task 3021, dataset 'sick'.
[2018-04-11 21:39:16] Started run 758 on task 3021, dataset 'sick'.
[2018-04-11 21:39:16] Error in run 751: Run already exists in server. Run id(s): {8956592}
[2018-04-11 21:39:16] Started run 759 on task 3021, dataset 'sick'.
[2018-04-11 21:39:17] Error in run 752: Run already exists in server. Run id(s): {8956591}
[2018-04-11 21:39:17] Started run 760 on task 3021, dataset 'sick'.
[2018-04-11 21:39:18] Error in run 753: Run already exists in server. Run id(s): {8956603}
[2018-04-11 21:39:18] Started run 761 on task 3021, dataset 'sick'.
[2018-04-11 21:39:18] Error in run 754: Run already exists in server. Run id(s): {8956597}
[2018-04-11 21:39:18] Started run 762 on task 3021, dataset 'sick'.
[2018-04-11 21:39:19] Error in run 755: Run already exists in server. Run id(s): {8956573}
[2018-04-11 21:39:19] Started run 763 on task 3021, dataset 'sick'.
[2018-04-11 21:39:20] Error in run 756: Run already exists in server. Run id(s): {8956612}
[2018-04-11 21:39:20] Started run 764 on task 3021, dataset 'sick'.
[2018-04-11 21:39:20] Error in run 757: Run already exists in server. Run id(s): {8956619}
[2018-04-11 21:39:20] Error in run 758: Run already exists in server. Run id(s): {8956601}
[2018-04-11 21:39:20] Started run 765 on task 3021, dataset 'sick'.
[2018-04-11 21:39:20] Started run 766 on task 3021, dataset 'sick'.
[2018-04-11 21:39:21] Error in run 759: Run already exists in server. Run id(s): {8956614}
[2018-04-11 21:39:21] Started run 767 on task 3021, dataset 'sick'.
[2018-04-11 21:39:21] Error in run 760: Run already exists in server. Run id(s): {8956624}
[2018-04-11 21:39:21] Started run 768 on task 3021, dataset 'sick'.
[2018-04-11 21:39:23] Error in run 762: Run already exists in server. Run id(s): {8956607}
[2018-04-11 21:39:23] Started run 769 on task 3021, dataset 'sick'.
[2018-04-11 21:39:23] Error in run 761: Run already exists in server. Run id(s): {8956615}
[2018-04-11 21:39:23] Started run 770 on task 3021, dataset 'sick'.
[2018-04-11 21:39:23] Error in run 763: Run already exists in server. Run id(s): {8956610}
[2018-04-11 21:39:23] Started run 771 on task 3021, dataset 'sick'.
[2018-04-11 21:39:24] Error in run 764: Run already exists in server. Run id(s): {8956604}
[2018-04-11 21:39:24] Started run 772 on task 3021, dataset 'sick'.
[2018-04-11 21:39:25] Error in run 766: Run already exists in server. Run id(s): {8956617}
[2018-04-11 21:39:25] Error in run 765: Run already exists in server. Run id(s): {8956616}
[2018-04-11 21:39:25] Started run 773 on task 3021, dataset 'sick'.
[2018-04-11 21:39:25] Started run 774 on task 3021, dataset 'sick'.
[2018-04-11 21:39:25] Error in run 767: Run already exists in server. Run id(s): {8956627}
[2018-04-11 21:39:25] Started run 775 on task 3021, dataset 'sick'.
[2018-04-11 21:39:26] Error in run 768: Run already exists in server. Run id(s): {8956632}
[2018-04-11 21:39:26] Started run 776 on task 3021, dataset 'sick'.
[2018-04-11 21:39:27] Error in run 769: Run already exists in server. Run id(s): {8956637}
[2018-04-11 21:39:27] Started run 777 on task 3021, dataset 'sick'.
[2018-04-11 21:39:27] Error in run 770: Run already exists in server. Run id(s): {8956621}
[2018-04-11 21:39:27] Started run 778 on task 3021, dataset 'sick'.
[2018-04-11 21:39:28] Error in run 771: Run already exists in server. Run id(s): {8956631}
[2018-04-11 21:39:28] Started run 779 on task 3021, dataset 'sick'.
[2018-04-11 21:39:29] Error in run 772: Run already exists in server. Run id(s): {8998008}
[2018-04-11 21:39:29] Started run 780 on task 3021, dataset 'sick'.
[2018-04-11 21:39:30] Error in run 773: Run already exists in server. Run id(s): {8998015}
[2018-04-11 21:39:30] Error in run 774: Run already exists in server. Run id(s): {8998002}
[2018-04-11 21:39:30] Started run 781 on task 3021, dataset 'sick'.
[2018-04-11 21:39:30] Started run 782 on task 3021, dataset 'sick'.
[2018-04-11 21:39:30] Error in run 775: Run already exists in server. Run id(s): {8998024}
[2018-04-11 21:39:30] Started run 783 on task 3021, dataset 'sick'.
[2018-04-11 21:39:30] Error in run 776: Run already exists in server. Run id(s): {8997983}
[2018-04-11 21:39:30] Started run 784 on task 3021, dataset 'sick'.
[2018-04-11 21:39:32] Error in run 777: Run already exists in server. Run id(s): {8998025}
[2018-04-11 21:39:32] Started run 785 on task 3021, dataset 'sick'.
[2018-04-11 21:39:32] Error in run 778: Run already exists in server. Run id(s): {8998007}
[2018-04-11 21:39:32] Started run 786 on task 3021, dataset 'sick'.
[2018-04-11 21:39:33] Error in run 779: Run already exists in server. Run id(s): {8998037}
[2018-04-11 21:39:33] Started run 787 on task 3021, dataset 'sick'.
[2018-04-11 21:39:33] Error in run 780: Run already exists in server. Run id(s): {8998043}
[2018-04-11 21:39:33] Started run 788 on task 3021, dataset 'sick'.
[2018-04-11 21:39:34] Error in run 782: Run already exists in server. Run id(s): {8998068}
[2018-04-11 21:39:34] Error in run 781: Run already exists in server. Run id(s): {8998064}
[2018-04-11 21:39:34] Started run 789 on task 3021, dataset 'sick'.
[2018-04-11 21:39:34] Started run 790 on task 3021, dataset 'sick'.
[2018-04-11 21:39:35] Error in run 783: Run already exists in server. Run id(s): {8998061}
[2018-04-11 21:39:35] Started run 791 on task 3021, dataset 'sick'.
[2018-04-11 21:39:35] Error in run 784: Run already exists in server. Run id(s): {8998067}
[2018-04-11 21:39:35] Started run 792 on task 3021, dataset 'sick'.
[2018-04-11 21:39:36] Error in run 785: Run already exists in server. Run id(s): {8998079}
[2018-04-11 21:39:36] Started run 793 on task 3021, dataset 'sick'.
[2018-04-11 21:39:37] Error in run 786: Run already exists in server. Run id(s): {8998060}
[2018-04-11 21:39:37] Started run 794 on task 3021, dataset 'sick'.
[2018-04-11 21:39:37] Error in run 787: Run already exists in server. Run id(s): {8998089}
[2018-04-11 21:39:37] Started run 795 on task 3021, dataset 'sick'.
[2018-04-11 21:39:38] Error in run 788: Run already exists in server. Run id(s): {8998107}
[2018-04-11 21:39:38] Started run 796 on task 3021, dataset 'sick'.
[2018-04-11 21:39:39] Error in run 790: Run already exists in server. Run id(s): {8998129}
[2018-04-11 21:39:39] Started run 797 on task 3021, dataset 'sick'.
[2018-04-11 21:39:39] Error in run 789: Run already exists in server. Run id(s): {8998125}
[2018-04-11 21:39:39] Started run 798 on task 3021, dataset 'sick'.
[2018-04-11 21:39:39] Error in run 792: Run already exists in server. Run id(s): {8998102}
[2018-04-11 21:39:39] Error in run 791: Run already exists in server. Run id(s): {8998117}
[2018-04-11 21:39:39] Started run 799 on task 3021, dataset 'sick'.
[2018-04-11 21:39:39] Started run 800 on task 3021, dataset 'sick'.
[2018-04-11 21:39:41] Error in run 793: Run already exists in server. Run id(s): {8998118}
[2018-04-11 21:39:41] Error in run 794: Run already exists in server. Run id(s): {8998136}
[2018-04-11 21:39:41] Started run 801 on task 3021, dataset 'sick'.
[2018-04-11 21:39:41] Started run 802 on task 3021, dataset 'sick'.
[2018-04-11 21:39:42] Error in run 795: Run already exists in server. Run id(s): {8998150}
[2018-04-11 21:39:42] Started run 803 on task 3021, dataset 'sick'.
[2018-04-11 21:39:43] Error in run 796: Run already exists in server. Run id(s): {8998156}
[2018-04-11 21:39:43] Started run 804 on task 3021, dataset 'sick'.
[2018-04-11 21:39:43] Error in run 798: Run already exists in server. Run id(s): {8956623}
[2018-04-11 21:39:43] Error in run 797: Run already exists in server. Run id(s): {8998161}
[2018-04-11 21:39:43] Started run 805 on task 3021, dataset 'sick'.
[2018-04-11 21:39:43] Started run 806 on task 3021, dataset 'sick'.
[2018-04-11 21:39:44] Error in run 799: Run already exists in server. Run id(s): {8998172}
[2018-04-11 21:39:44] Started run 807 on task 3021, dataset 'sick'.
[2018-04-11 21:39:44] Error in run 800: Run already exists in server. Run id(s): {8998185}
[2018-04-11 21:39:44] Started run 808 on task 3021, dataset 'sick'.
[2018-04-11 21:39:46] Error in run 801: Run already exists in server. Run id(s): {8998174}
[2018-04-11 21:39:46] Started run 809 on task 3021, dataset 'sick'.
[2018-04-11 21:39:46] Error in run 802: Run already exists in server. Run id(s): {8998187}
[2018-04-11 21:39:46] Started run 810 on task 3021, dataset 'sick'.
[2018-04-11 21:39:47] Error in run 803: Run already exists in server. Run id(s): {8956650}
[2018-04-11 21:39:47] Started run 811 on task 3021, dataset 'sick'.
[2018-04-11 21:39:47] Error in run 804: Run already exists in server. Run id(s): {8998197}
[2018-04-11 21:39:47] Started run 812 on task 3021, dataset 'sick'.
[2018-04-11 21:39:48] Error in run 806: Run already exists in server. Run id(s): {8998207}
[2018-04-11 21:39:48] Started run 813 on task 3021, dataset 'sick'.
[2018-04-11 21:39:48] Error in run 805: Run already exists in server. Run id(s): {8956638}
[2018-04-11 21:39:48] Started run 814 on task 3021, dataset 'sick'.
[2018-04-11 21:39:49] Error in run 807: Run already exists in server. Run id(s): {8998208}
[2018-04-11 21:39:49] Error in run 808: Run already exists in server. Run id(s): {8998212}
[2018-04-11 21:39:49] Started run 815 on task 3021, dataset 'sick'.
[2018-04-11 21:39:49] Started run 816 on task 3021, dataset 'sick'.
[2018-04-11 21:39:50] Error in run 809: Run already exists in server. Run id(s): {8998227}
[2018-04-11 21:39:50] Started run 817 on task 3021, dataset 'sick'.
[2018-04-11 21:39:51] Error in run 810: Run already exists in server. Run id(s): {8998236}
[2018-04-11 21:39:51] Started run 818 on task 3021, dataset 'sick'.
[2018-04-11 21:39:51] Error in run 811: Run already exists in server. Run id(s): {8998231}
[2018-04-11 21:39:51] Started run 819 on task 3021, dataset 'sick'.
[2018-04-11 21:39:52] Error in run 812: Run already exists in server. Run id(s): {8998232}
[2018-04-11 21:39:52] Started run 820 on task 3021, dataset 'sick'.
[2018-04-11 21:39:53] Error in run 813: Run already exists in server. Run id(s): {8998254}
[2018-04-11 21:39:53] Error in run 814: Run already exists in server. Run id(s): {8998266}
[2018-04-11 21:39:53] Started run 821 on task 3021, dataset 'sick'.
[2018-04-11 21:39:53] Started run 822 on task 3021, dataset 'sick'.
[2018-04-11 21:39:54] Error in run 816: Run already exists in server. Run id(s): {8956639}
[2018-04-11 21:39:54] Started run 823 on task 3021, dataset 'sick'.
[2018-04-11 21:39:54] Error in run 815: Run already exists in server. Run id(s): {8956673}
[2018-04-11 21:39:54] Started run 824 on task 3021, dataset 'sick'.
[2018-04-11 21:39:55] Error in run 817: Run already exists in server. Run id(s): {8998268}
[2018-04-11 21:39:55] Started run 825 on task 3021, dataset 'sick'.
[2018-04-11 21:39:55] Error in run 818: Run already exists in server. Run id(s): {8998276}
[2018-04-11 21:39:55] Started run 826 on task 3021, dataset 'sick'.
[2018-04-11 21:39:56] Error in run 819: Run already exists in server. Run id(s): {8998261}
[2018-04-11 21:39:56] Started run 827 on task 3021, dataset 'sick'.
[2018-04-11 21:39:57] Error in run 820: Run already exists in server. Run id(s): {8998283}
[2018-04-11 21:39:57] Started run 828 on task 3021, dataset 'sick'.
[2018-04-11 21:39:58] Error in run 821: Run already exists in server. Run id(s): {8998291}
[2018-04-11 21:39:58] Error in run 822: Run already exists in server. Run id(s): {8956611}
[2018-04-11 21:39:58] Started run 829 on task 3021, dataset 'sick'.
[2018-04-11 21:39:58] Started run 830 on task 3021, dataset 'sick'.
[2018-04-11 21:39:58] Error in run 824: Run already exists in server. Run id(s): {8956635}
[2018-04-11 21:39:58] Error in run 823: Run already exists in server. Run id(s): {8998277}
[2018-04-11 21:39:58] Started run 831 on task 3021, dataset 'sick'.
[2018-04-11 21:39:58] Started run 832 on task 3021, dataset 'sick'.
[2018-04-11 21:39:59] Error in run 825: Run already exists in server. Run id(s): {8956618}
[2018-04-11 21:39:59] Started run 833 on task 3021, dataset 'sick'.
[2018-04-11 21:40:00] Error in run 826: Run already exists in server. Run id(s): {8956674}
[2018-04-11 21:40:00] Started run 834 on task 3021, dataset 'sick'.
[2018-04-11 21:40:00] Error in run 827: Run already exists in server. Run id(s): {8956667}
[2018-04-11 21:40:00] Started run 835 on task 3021, dataset 'sick'.
[2018-04-11 21:40:02] Error in run 828: Run already exists in server. Run id(s): {8956679}
[2018-04-11 21:40:02] Started run 836 on task 3021, dataset 'sick'.
[2018-04-11 21:40:02] Error in run 829: Run already exists in server. Run id(s): {8998302}
[2018-04-11 21:40:02] Started run 837 on task 3021, dataset 'sick'.
[2018-04-11 21:40:02] Error in run 830: Run already exists in server. Run id(s): {8956645}
[2018-04-11 21:40:02] Started run 838 on task 3021, dataset 'sick'.
[2018-04-11 21:40:03] Error in run 831: Run already exists in server. Run id(s): {8956653}
[2018-04-11 21:40:03] Started run 839 on task 3021, dataset 'sick'.
[2018-04-11 21:40:03] Error in run 832: Run already exists in server. Run id(s): {8956662}
[2018-04-11 21:40:03] Started run 840 on task 3021, dataset 'sick'.
[2018-04-11 21:40:04] Error in run 833: Run already exists in server. Run id(s): {8956672}
[2018-04-11 21:40:04] Started run 841 on task 3021, dataset 'sick'.
[2018-04-11 21:40:05] Error in run 834: Run already exists in server. Run id(s): {8998323}
[2018-04-11 21:40:05] Started run 842 on task 3021, dataset 'sick'.
[2018-04-11 21:40:05] Error in run 835: Run already exists in server. Run id(s): {8956642}
[2018-04-11 21:40:05] Started run 843 on task 3021, dataset 'sick'.
[2018-04-11 21:40:06] Error in run 836: Run already exists in server. Run id(s): {8956656}
[2018-04-11 21:40:06] Started run 844 on task 3021, dataset 'sick'.
[2018-04-11 21:40:07] Error in run 837: Run already exists in server. Run id(s): {8956655}
[2018-04-11 21:40:07] Started run 845 on task 3021, dataset 'sick'.
[2018-04-11 21:40:07] Error in run 838: Run already exists in server. Run id(s): {8956657}
[2018-04-11 21:40:07] Started run 846 on task 3021, dataset 'sick'.
[2018-04-11 21:40:07] Error in run 839: Run already exists in server. Run id(s): {8956663}
[2018-04-11 21:40:07] Started run 847 on task 3021, dataset 'sick'.
[2018-04-11 21:40:07] Error in run 840: Run already exists in server. Run id(s): {8956660}
[2018-04-11 21:40:07] Started run 848 on task 3021, dataset 'sick'.
[2018-04-11 21:40:08] Error in run 841: Run already exists in server. Run id(s): {8956682}
[2018-04-11 21:40:08] Started run 849 on task 3021, dataset 'sick'.
[2018-04-11 21:40:10] Error in run 842: Run already exists in server. Run id(s): {8956669}
[2018-04-11 21:40:10] Started run 850 on task 3021, dataset 'sick'.
[2018-04-11 21:40:10] Error in run 843: Run already exists in server. Run id(s): {8956678}
[2018-04-11 21:40:10] Started run 851 on task 3021, dataset 'sick'.
[2018-04-11 21:40:11] Error in run 844: Run already exists in server. Run id(s): {8956661}
[2018-04-11 21:40:11] Started run 852 on task 3021, dataset 'sick'.
[2018-04-11 21:40:11] Error in run 845: Run already exists in server. Run id(s): {8956643}
[2018-04-11 21:40:11] Started run 853 on task 3021, dataset 'sick'.
[2018-04-11 21:40:12] Error in run 846: Run already exists in server. Run id(s): {8956648}
[2018-04-11 21:40:12] Started run 854 on task 3021, dataset 'sick'.
[2018-04-11 21:40:12] Error in run 847: Run already exists in server. Run id(s): {8998338}
[2018-04-11 21:40:12] Started run 855 on task 3021, dataset 'sick'.
[2018-04-11 21:40:12] Error in run 848: Run already exists in server. Run id(s): {8956676}
[2018-04-11 21:40:12] Started run 856 on task 3021, dataset 'sick'.
[2018-04-11 21:40:13] Error in run 849: Run already exists in server. Run id(s): {8956644}
[2018-04-11 21:40:13] Started run 857 on task 3021, dataset 'sick'.
[2018-04-11 21:40:14] Error in run 851: Run already exists in server. Run id(s): {8956677}
[2018-04-11 21:40:14] Error in run 850: Run already exists in server. Run id(s): {8956622}
[2018-04-11 21:40:14] Started run 858 on task 3021, dataset 'sick'.
[2018-04-11 21:40:14] Started run 859 on task 3021, dataset 'sick'.
[2018-04-11 21:40:15] Error in run 852: Run already exists in server. Run id(s): {8956628}
[2018-04-11 21:40:15] Started run 860 on task 3021, dataset 'sick'.
[2018-04-11 21:40:16] Error in run 853: Run already exists in server. Run id(s): {8956647}
[2018-04-11 21:40:16] Started run 861 on task 3021, dataset 'sick'.
[2018-04-11 21:40:16] Error in run 854: Run already exists in server. Run id(s): {8956668}
[2018-04-11 21:40:16] Started run 862 on task 3021, dataset 'sick'.
[2018-04-11 21:40:17] Error in run 856: Run already exists in server. Run id(s): {8956629}
[2018-04-11 21:40:17] Error in run 855: Run already exists in server. Run id(s): {8956626}
[2018-04-11 21:40:17] Started run 863 on task 3021, dataset 'sick'.
[2018-04-11 21:40:17] Started run 864 on task 3021, dataset 'sick'.
[2018-04-11 21:40:18] Error in run 857: Run already exists in server. Run id(s): {8956664}
[2018-04-11 21:40:18] Started run 865 on task 3021, dataset 'sick'.
[2018-04-11 21:40:19] Error in run 859: Run already exists in server. Run id(s): {8956652}
[2018-04-11 21:40:19] Started run 866 on task 3021, dataset 'sick'.
[2018-04-11 21:40:19] Error in run 858: Run already exists in server. Run id(s): {8998342}
[2018-04-11 21:40:19] Started run 867 on task 3021, dataset 'sick'.
[2018-04-11 21:40:20] Error in run 860: Run already exists in server. Run id(s): {8956685}
[2018-04-11 21:40:20] Started run 868 on task 3021, dataset 'sick'.
[2018-04-11 21:40:21] Error in run 861: Run already exists in server. Run id(s): {8956665}
[2018-04-11 21:40:21] Started run 869 on task 3021, dataset 'sick'.
[2018-04-11 21:40:21] Error in run 862: Run already exists in server. Run id(s): {8956666}
[2018-04-11 21:40:21] Started run 870 on task 3021, dataset 'sick'.
[2018-04-11 21:40:21] Error in run 864: Run already exists in server. Run id(s): {8956694}
[2018-04-11 21:40:21] Error in run 863: Run already exists in server. Run id(s): {8956630}
[2018-04-11 21:40:21] Started run 871 on task 3021, dataset 'sick'.
[2018-04-11 21:40:21] Started run 872 on task 3021, dataset 'sick'.
[2018-04-11 21:40:22] Error in run 865: Run already exists in server. Run id(s): {8956681}
[2018-04-11 21:40:22] Started run 873 on task 3021, dataset 'sick'.
[2018-04-11 21:40:24] Error in run 867: Run already exists in server. Run id(s): {8956646}
[2018-04-11 21:40:24] Started run 874 on task 3021, dataset 'sick'.
[2018-04-11 21:40:24] Error in run 866: Run already exists in server. Run id(s): {8956684}
[2018-04-11 21:40:24] Started run 875 on task 3021, dataset 'sick'.
[2018-04-11 21:40:25] Error in run 868: Run already exists in server. Run id(s): {8956687}
[2018-04-11 21:40:25] Started run 876 on task 3021, dataset 'sick'.
[2018-04-11 21:40:25] Error in run 869: Run already exists in server. Run id(s): {8956683}
[2018-04-11 21:40:25] Error in run 870: Run already exists in server. Run id(s): {8956690}
[2018-04-11 21:40:25] Started run 877 on task 3021, dataset 'sick'.
[2018-04-11 21:40:25] Started run 878 on task 3021, dataset 'sick'.
[2018-04-11 21:40:26] Error in run 872: Run already exists in server. Run id(s): {8998347}
[2018-04-11 21:40:26] Started run 879 on task 3021, dataset 'sick'.
[2018-04-11 21:40:26] Error in run 871: Run already exists in server. Run id(s): {8956686}
[2018-04-11 21:40:26] Started run 880 on task 3021, dataset 'sick'.
[2018-04-11 21:40:27] Error in run 873: Run already exists in server. Run id(s): {8956649}
[2018-04-11 21:40:27] Started run 881 on task 3021, dataset 'sick'.
[2018-04-11 21:40:29] Error in run 874: Run already exists in server. Run id(s): {8956692}
[2018-04-11 21:40:29] Error in run 875: Run already exists in server. Run id(s): {8956675}
[2018-04-11 21:40:29] Started run 882 on task 3021, dataset 'sick'.
[2018-04-11 21:40:29] Started run 883 on task 3021, dataset 'sick'.
[2018-04-11 21:40:29] Error in run 876: Run already exists in server. Run id(s): {8956705}
[2018-04-11 21:40:29] Started run 884 on task 3021, dataset 'sick'.
[2018-04-11 21:40:30] Error in run 878: Run already exists in server. Run id(s): {8998345}
[2018-04-11 21:40:30] Error in run 877: Run already exists in server. Run id(s): {8998349}
[2018-04-11 21:40:30] Started run 885 on task 3021, dataset 'sick'.
[2018-04-11 21:40:30] Started run 886 on task 3021, dataset 'sick'.
[2018-04-11 21:40:31] Error in run 879: Run already exists in server. Run id(s): {8956698}
[2018-04-11 21:40:31] Started run 887 on task 3021, dataset 'sick'.
[2018-04-11 21:40:31] Error in run 880: Run already exists in server. Run id(s): {8956701}
[2018-04-11 21:40:31] Started run 888 on task 3021, dataset 'sick'.
[2018-04-11 21:40:31] Error in run 881: Run already exists in server. Run id(s): {8956689}
[2018-04-11 21:40:31] Started run 889 on task 3021, dataset 'sick'.
[2018-04-11 21:40:33] Error in run 882: Run already exists in server. Run id(s): {8956695}
[2018-04-11 21:40:33] Error in run 883: Run already exists in server. Run id(s): {8956706}
[2018-04-11 21:40:33] Started run 890 on task 3021, dataset 'sick'.
[2018-04-11 21:40:33] Started run 891 on task 3021, dataset 'sick'.
[2018-04-11 21:40:34] Error in run 884: Run already exists in server. Run id(s): {8956709}
[2018-04-11 21:40:34] Started run 892 on task 3021, dataset 'sick'.
[2018-04-11 21:40:35] Error in run 885: Run already exists in server. Run id(s): {8956704}
[2018-04-11 21:40:35] Started run 893 on task 3021, dataset 'sick'.
[2018-04-11 21:40:35] Error in run 886: Run already exists in server. Run id(s): {8956717}
[2018-04-11 21:40:35] Started run 894 on task 3021, dataset 'sick'.
[2018-04-11 21:40:35] Error in run 887: Run already exists in server. Run id(s): {8956711}
[2018-04-11 21:40:35] Started run 895 on task 3021, dataset 'sick'.
[2018-04-11 21:40:36] Error in run 888: Run already exists in server. Run id(s): {8956702}
[2018-04-11 21:40:36] Started run 896 on task 3021, dataset 'sick'.
[2018-04-11 21:40:36] Error in run 889: Run already exists in server. Run id(s): {8956716}
[2018-04-11 21:40:36] Started run 897 on task 3021, dataset 'sick'.
[2018-04-11 21:40:38] Error in run 891: Run already exists in server. Run id(s): {8956693}
[2018-04-11 21:40:38] Error in run 890: Run already exists in server. Run id(s): {8956700}
[2018-04-11 21:40:38] Started run 898 on task 3021, dataset 'sick'.
[2018-04-11 21:40:38] Started run 899 on task 3021, dataset 'sick'.
[2018-04-11 21:40:39] Error in run 892: Run already exists in server. Run id(s): {8956671}
[2018-04-11 21:40:39] Started run 900 on task 3021, dataset 'sick'.
[2018-04-11 21:40:39] Error in run 893: Run already exists in server. Run id(s): {8956699}
[2018-04-11 21:40:39] Started run 901 on task 3021, dataset 'sick'.
[2018-04-11 21:40:40] Error in run 894: Run already exists in server. Run id(s): {8956707}
[2018-04-11 21:40:40] Started run 902 on task 3021, dataset 'sick'.
[2018-04-11 21:40:40] Error in run 895: Run already exists in server. Run id(s): {8956722}
[2018-04-11 21:40:40] Started run 903 on task 3021, dataset 'sick'.
[2018-04-11 21:40:40] Error in run 896: Run already exists in server. Run id(s): {8956710}
[2018-04-11 21:40:40] Started run 904 on task 3021, dataset 'sick'.
[2018-04-11 21:40:40] Error in run 897: Run already exists in server. Run id(s): {8956720}
[2018-04-11 21:40:40] Started run 905 on task 3021, dataset 'sick'.
[2018-04-11 21:40:43] Error in run 899: Run already exists in server. Run id(s): {8956728}
[2018-04-11 21:40:43] Started run 906 on task 3021, dataset 'sick'.
[2018-04-11 21:40:43] Error in run 898: Run already exists in server. Run id(s): {8956708}
[2018-04-11 21:40:43] Started run 907 on task 3021, dataset 'sick'.
[2018-04-11 21:40:44] Error in run 900: Run already exists in server. Run id(s): {8956719}
[2018-04-11 21:40:44] Started run 908 on task 3021, dataset 'sick'.
[2018-04-11 21:40:44] Error in run 901: Run already exists in server. Run id(s): {8956737}
[2018-04-11 21:40:44] Started run 909 on task 3021, dataset 'sick'.
[2018-04-11 21:40:44] Error in run 903: Run already exists in server. Run id(s): {8956725}
[2018-04-11 21:40:44] Error in run 902: Run already exists in server. Run id(s): {8956731}
[2018-04-11 21:40:44] Started run 910 on task 3021, dataset 'sick'.
[2018-04-11 21:40:44] Started run 911 on task 3021, dataset 'sick'.
[2018-04-11 21:40:45] Error in run 904: Run already exists in server. Run id(s): {8956697}
[2018-04-11 21:40:45] Error in run 905: Run already exists in server. Run id(s): {8956696}
[2018-04-11 21:40:45] Started run 912 on task 3021, dataset 'sick'.
[2018-04-11 21:40:45] Started run 913 on task 3021, dataset 'sick'.
[2018-04-11 21:40:47] Error in run 906: Run already exists in server. Run id(s): {8956715}
[2018-04-11 21:40:47] Started run 914 on task 3021, dataset 'sick'.
[2018-04-11 21:40:48] Error in run 907: Run already exists in server. Run id(s): {8956734}
[2018-04-11 21:40:48] Started run 915 on task 3021, dataset 'sick'.
[2018-04-11 21:40:48] Error in run 909: Run already exists in server. Run id(s): {8956738}
[2018-04-11 21:40:48] Error in run 908: Run already exists in server. Run id(s): {8956714}
[2018-04-11 21:40:48] Started run 916 on task 3021, dataset 'sick'.
[2018-04-11 21:40:48] Started run 917 on task 3021, dataset 'sick'.
[2018-04-11 21:40:49] Error in run 911: Run already exists in server. Run id(s): {8998419}
[2018-04-11 21:40:49] Started run 918 on task 3021, dataset 'sick'.
[2018-04-11 21:40:49] Error in run 910: Run already exists in server. Run id(s): {8956729}
[2018-04-11 21:40:49] Started run 919 on task 3021, dataset 'sick'.
[2018-04-11 21:40:50] Error in run 912: Run already exists in server. Run id(s): {8998405}
[2018-04-11 21:40:50] Started run 920 on task 3021, dataset 'sick'.
[2018-04-11 21:40:50] Error in run 913: Run already exists in server. Run id(s): {8998421}
[2018-04-11 21:40:50] Started run 921 on task 3021, dataset 'sick'.
[2018-04-11 21:40:52] Error in run 914: Run already exists in server. Run id(s): {8998427}
[2018-04-11 21:40:52] Started run 922 on task 3021, dataset 'sick'.
[2018-04-11 21:40:52] Error in run 915: Run already exists in server. Run id(s): {8998432}
[2018-04-11 21:40:52] Started run 923 on task 3021, dataset 'sick'.
[2018-04-11 21:40:53] Error in run 917: Run already exists in server. Run id(s): {8998418}
[2018-04-11 21:40:53] Error in run 916: Run already exists in server. Run id(s): {8998431}
[2018-04-11 21:40:53] Started run 924 on task 3021, dataset 'sick'.
[2018-04-11 21:40:53] Started run 925 on task 3021, dataset 'sick'.
[2018-04-11 21:40:54] Error in run 918: Run already exists in server. Run id(s): {8998430}
[2018-04-11 21:40:54] Started run 926 on task 3021, dataset 'sick'.
[2018-04-11 21:40:54] Error in run 919: Run already exists in server. Run id(s): {8998482}
[2018-04-11 21:40:54] Started run 927 on task 3021, dataset 'sick'.
[2018-04-11 21:40:55] Error in run 921: Run already exists in server. Run id(s): {8998469}
[2018-04-11 21:40:55] Started run 928 on task 3021, dataset 'sick'.
[2018-04-11 21:40:55] Error in run 920: Run already exists in server. Run id(s): {8998485}
[2018-04-11 21:40:55] Started run 929 on task 3021, dataset 'sick'.
[2018-04-11 21:40:56] Error in run 922: Run already exists in server. Run id(s): {8998474}
[2018-04-11 21:40:56] Started run 930 on task 3021, dataset 'sick'.
[2018-04-11 21:40:57] Error in run 923: Run already exists in server. Run id(s): {8998484}
[2018-04-11 21:40:57] Started run 931 on task 3021, dataset 'sick'.
[2018-04-11 21:40:57] Error in run 925: Run already exists in server. Run id(s): {8998499}
[2018-04-11 21:40:57] Error in run 924: Run already exists in server. Run id(s): {8998464}
[2018-04-11 21:40:57] Started run 932 on task 3021, dataset 'sick'.
[2018-04-11 21:40:57] Started run 933 on task 3021, dataset 'sick'.
[2018-04-11 21:40:58] Error in run 926: Run already exists in server. Run id(s): {8998486}
[2018-04-11 21:40:58] Started run 934 on task 3021, dataset 'sick'.
[2018-04-11 21:40:58] Error in run 927: Run already exists in server. Run id(s): {8998522}
[2018-04-11 21:40:58] Started run 935 on task 3021, dataset 'sick'.
[2018-04-11 21:40:59] Error in run 928: Run already exists in server. Run id(s): {8998511}
[2018-04-11 21:40:59] Error in run 929: Run already exists in server. Run id(s): {8998554}
[2018-04-11 21:40:59] Started run 936 on task 3021, dataset 'sick'.
[2018-04-11 21:40:59] Started run 937 on task 3021, dataset 'sick'.
[2018-04-11 21:41:01] Error in run 930: Run already exists in server. Run id(s): {8998547}
[2018-04-11 21:41:01] Started run 938 on task 3021, dataset 'sick'.
[2018-04-11 21:41:01] Error in run 931: Run already exists in server. Run id(s): {8998552}
[2018-04-11 21:41:01] Started run 939 on task 3021, dataset 'sick'.
[2018-04-11 21:41:02] Error in run 932: Run already exists in server. Run id(s): {8998566}
[2018-04-11 21:41:02] Started run 940 on task 3021, dataset 'sick'.
[2018-04-11 21:41:02] Error in run 933: Run already exists in server. Run id(s): {8998542}
[2018-04-11 21:41:02] Started run 941 on task 3021, dataset 'sick'.
[2018-04-11 21:41:03] Error in run 934: Run already exists in server. Run id(s): {8998532}
[2018-04-11 21:41:03] Started run 942 on task 3021, dataset 'sick'.
[2018-04-11 21:41:03] Error in run 935: Run already exists in server. Run id(s): {8998578}
[2018-04-11 21:41:03] Started run 943 on task 3021, dataset 'sick'.
[2018-04-11 21:41:03] Error in run 937: Run already exists in server. Run id(s): {8998594}
[2018-04-11 21:41:03] Error in run 936: Run already exists in server. Run id(s): {8998588}
[2018-04-11 21:41:03] Started run 944 on task 3021, dataset 'sick'.
[2018-04-11 21:41:03] Started run 945 on task 3021, dataset 'sick'.
[2018-04-11 21:41:06] Error in run 938: Run already exists in server. Run id(s): {8998611}
[2018-04-11 21:41:06] Started run 946 on task 3021, dataset 'sick'.
[2018-04-11 21:41:06] Error in run 939: Run already exists in server. Run id(s): {8998612}
[2018-04-11 21:41:06] Started run 947 on task 3021, dataset 'sick'.
[2018-04-11 21:41:06] Error in run 941: Run already exists in server. Run id(s): {8998610}
[2018-04-11 21:41:06] Started run 948 on task 3021, dataset 'sick'.
[2018-04-11 21:41:06] Error in run 940: Run already exists in server. Run id(s): {8998601}
[2018-04-11 21:41:06] Started run 949 on task 3021, dataset 'sick'.
[2018-04-11 21:41:07] Error in run 942: Run already exists in server. Run id(s): {8998621}
[2018-04-11 21:41:07] Started run 950 on task 3021, dataset 'sick'.
[2018-04-11 21:41:07] Error in run 943: Run already exists in server. Run id(s): {8998640}
[2018-04-11 21:41:07] Started run 951 on task 3021, dataset 'sick'.
[2018-04-11 21:41:08] Error in run 945: Run already exists in server. Run id(s): {8998650}
[2018-04-11 21:41:08] Error in run 944: Run already exists in server. Run id(s): {8998636}
[2018-04-11 21:41:08] Started run 952 on task 3021, dataset 'sick'.
[2018-04-11 21:41:08] Started run 953 on task 3021, dataset 'sick'.
[2018-04-11 21:41:10] Error in run 946: Run already exists in server. Run id(s): {8998666}
[2018-04-11 21:41:10] Started run 954 on task 3021, dataset 'sick'.
[2018-04-11 21:41:11] Error in run 947: Run already exists in server. Run id(s): {8998668}
[2018-04-11 21:41:11] Started run 955 on task 3021, dataset 'sick'.
[2018-04-11 21:41:11] Error in run 948: Run already exists in server. Run id(s): {8998674}
[2018-04-11 21:41:11] Error in run 949: Run already exists in server. Run id(s): {8998652}
[2018-04-11 21:41:11] Started run 956 on task 3021, dataset 'sick'.
[2018-04-11 21:41:11] Started run 957 on task 3021, dataset 'sick'.
[2018-04-11 21:41:12] Error in run 950: Run already exists in server. Run id(s): {8998679}
[2018-04-11 21:41:12] Started run 958 on task 3021, dataset 'sick'.
[2018-04-11 21:41:12] Error in run 951: Run already exists in server. Run id(s): {8998678}
[2018-04-11 21:41:12] Started run 959 on task 3021, dataset 'sick'.
[2018-04-11 21:41:13] Error in run 952: Run already exists in server. Run id(s): {8998700}
[2018-04-11 21:41:13] Started run 960 on task 3021, dataset 'sick'.
[2018-04-11 21:41:13] Error in run 953: Run already exists in server. Run id(s): {8956741}
[2018-04-11 21:41:13] Started run 961 on task 3021, dataset 'sick'.
[2018-04-11 21:41:15] Error in run 954: Run already exists in server. Run id(s): {8998714}
[2018-04-11 21:41:15] Started run 962 on task 3021, dataset 'sick'.
[2018-04-11 21:41:15] Error in run 955: Run already exists in server. Run id(s): {8956750}
[2018-04-11 21:41:15] Started run 963 on task 3021, dataset 'sick'.
[2018-04-11 21:41:15] Error in run 957: Run already exists in server. Run id(s): {8998707}
[2018-04-11 21:41:15] Error in run 956: Run already exists in server. Run id(s): {8956747}
[2018-04-11 21:41:15] Started run 964 on task 3021, dataset 'sick'.
[2018-04-11 21:41:15] Started run 965 on task 3021, dataset 'sick'.
[2018-04-11 21:41:16] Error in run 959: Run already exists in server. Run id(s): {8998727}
[2018-04-11 21:41:16] Started run 966 on task 3021, dataset 'sick'.
[2018-04-11 21:41:16] Error in run 958: Run already exists in server. Run id(s): {8998731}
[2018-04-11 21:41:16] Started run 967 on task 3021, dataset 'sick'.
[2018-04-11 21:41:17] Error in run 960: Run already exists in server. Run id(s): {8956732}
[2018-04-11 21:41:17] Started run 968 on task 3021, dataset 'sick'.
[2018-04-11 21:41:17] Error in run 961: Run already exists in server. Run id(s): {8998713}
[2018-04-11 21:41:17] Started run 969 on task 3021, dataset 'sick'.
[2018-04-11 21:41:19] Error in run 962: Run already exists in server. Run id(s): {8998741}
[2018-04-11 21:41:19] Started run 970 on task 3021, dataset 'sick'.
[2018-04-11 21:41:20] Error in run 965: Run already exists in server. Run id(s): {8998775}
[2018-04-11 21:41:20] Error in run 963: Run already exists in server. Run id(s): {8998747}
[2018-04-11 21:41:20] Started run 971 on task 3021, dataset 'sick'.
[2018-04-11 21:41:20] Error in run 964: Run already exists in server. Run id(s): {8998766}
[2018-04-11 21:41:20] Started run 973 on task 3021, dataset 'sick'.
[2018-04-11 21:41:20] Started run 972 on task 3021, dataset 'sick'.
[2018-04-11 21:41:21] Error in run 966: Run already exists in server. Run id(s): {8998774}
[2018-04-11 21:41:21] Error in run 967: Run already exists in server. Run id(s): {8956739}
[2018-04-11 21:41:21] Started run 974 on task 3021, dataset 'sick'.
[2018-04-11 21:41:21] Started run 975 on task 3021, dataset 'sick'.
[2018-04-11 21:41:22] Error in run 968: Run already exists in server. Run id(s): {8956743}
[2018-04-11 21:41:22] Started run 976 on task 3021, dataset 'sick'.
[2018-04-11 21:41:22] Error in run 969: Run already exists in server. Run id(s): {8998799}
[2018-04-11 21:41:22] Started run 977 on task 3021, dataset 'sick'.
[2018-04-11 21:41:24] Error in run 970: Run already exists in server. Run id(s): {8956748}
[2018-04-11 21:41:24] Started run 978 on task 3021, dataset 'sick'.
[2018-04-11 21:41:25] Error in run 971: Run already exists in server. Run id(s): {8956726}
[2018-04-11 21:41:25] Started run 979 on task 3021, dataset 'sick'.
[2018-04-11 21:41:25] Error in run 972: Run already exists in server. Run id(s): {8956742}
[2018-04-11 21:41:25] Started run 980 on task 3021, dataset 'sick'.
[2018-04-11 21:41:25] Error in run 973: Run already exists in server. Run id(s): {8998796}
[2018-04-11 21:41:25] Started run 981 on task 3021, dataset 'sick'.
[2018-04-11 21:41:26] Error in run 974: Run already exists in server. Run id(s): {8998808}
[2018-04-11 21:41:26] Error in run 975: Run already exists in server. Run id(s): {8998802}
[2018-04-11 21:41:26] Started run 982 on task 3021, dataset 'sick'.
[2018-04-11 21:41:26] Started run 983 on task 3021, dataset 'sick'.
[2018-04-11 21:41:26] Error in run 976: Run already exists in server. Run id(s): {8998806}
[2018-04-11 21:41:26] Started run 984 on task 3021, dataset 'sick'.
[2018-04-11 21:41:26] Error in run 977: Run already exists in server. Run id(s): {8956724}
[2018-04-11 21:41:26] Started run 985 on task 3021, dataset 'sick'.
[2018-04-11 21:41:29] Error in run 978: Run already exists in server. Run id(s): {8956749}
[2018-04-11 21:41:29] Started run 986 on task 3021, dataset 'sick'.
[2018-04-11 21:41:29] Error in run 979: Run already exists in server. Run id(s): {8998832}
[2018-04-11 21:41:29] Started run 987 on task 3021, dataset 'sick'.
[2018-04-11 21:41:30] Error in run 981: Run already exists in server. Run id(s): {8998830}
[2018-04-11 21:41:30] Error in run 980: Run already exists in server. Run id(s): {8956764}
[2018-04-11 21:41:30] Started run 988 on task 3021, dataset 'sick'.
[2018-04-11 21:41:30] Started run 989 on task 3021, dataset 'sick'.
[2018-04-11 21:41:30] Error in run 983: Run already exists in server. Run id(s): {8956721}
[2018-04-11 21:41:30] Started run 990 on task 3021, dataset 'sick'.
[2018-04-11 21:41:30] Error in run 982: Run already exists in server. Run id(s): {8956753}
[2018-04-11 21:41:30] Started run 991 on task 3021, dataset 'sick'.
[2018-04-11 21:41:31] Error in run 984: Run already exists in server. Run id(s): {8956760}
[2018-04-11 21:41:31] Started run 992 on task 3021, dataset 'sick'.
[2018-04-11 21:41:31] Error in run 985: Run already exists in server. Run id(s): {8956745}
[2018-04-11 21:41:31] Started run 993 on task 3021, dataset 'sick'.
[2018-04-11 21:41:33] Error in run 986: Run already exists in server. Run id(s): {8956740}
[2018-04-11 21:41:33] Started run 994 on task 3021, dataset 'sick'.
[2018-04-11 21:41:34] Error in run 987: Run already exists in server. Run id(s): {8956730}
[2018-04-11 21:41:34] Started run 995 on task 3021, dataset 'sick'.
[2018-04-11 21:41:34] Error in run 988: Run already exists in server. Run id(s): {8956755}
[2018-04-11 21:41:34] Started run 996 on task 3021, dataset 'sick'.
[2018-04-11 21:41:34] Error in run 989: Run already exists in server. Run id(s): {8956756}
[2018-04-11 21:41:34] Started run 997 on task 3021, dataset 'sick'.
[2018-04-11 21:41:35] Error in run 991: Run already exists in server. Run id(s): {8956746}
[2018-04-11 21:41:35] Started run 998 on task 3021, dataset 'sick'.
[2018-04-11 21:41:35] Error in run 990: Run already exists in server. Run id(s): {8956744}
[2018-04-11 21:41:35] Started run 999 on task 3021, dataset 'sick'.
[2018-04-11 21:41:36] Error in run 992: Run already exists in server. Run id(s): {8956762}
[2018-04-11 21:41:36] Error in run 993: Run already exists in server. Run id(s): {8956752}
[2018-04-11 21:41:36] Started run 1000 on task 3021, dataset 'sick'.
[2018-04-11 21:41:38] Error in run 994: Run already exists in server. Run id(s): {8956767}
[2018-04-11 21:41:38] Error in run 995: Run already exists in server. Run id(s): {8956766}
[2018-04-11 21:41:39] Error in run 997: Run already exists in server. Run id(s): {8956735}
[2018-04-11 21:41:39] Error in run 996: Run already exists in server. Run id(s): {8956769}
[2018-04-11 21:41:40] Error in run 998: Run already exists in server. Run id(s): {8956765}
[2018-04-11 21:41:40] Error in run 999: Run already exists in server. Run id(s): {8956763}
[2018-04-11 21:41:41] Error in run 1000: Run already exists in server. Run id(s): {8956757}
